---
title: "Cancer Genomics: Integrative and Scalable Solutions in R/Bioconductor"
---


## Abstract

This proposal develops scalable R / Bioconductor software infrastructure and data resources to integrate complex, heterogeneous, and large cancer genomic experiments. The falling cost of genomic assays facilitates collection of multiple data types (e.g., gene and transcript expression, structural variation, copy number, methylation, and microRNA data) from a set of clinical specimens. Furthermore, substantial resources are now available from large consortium activities like The Cancer Genome Atlas (TCGA). Existing analysis pipelines focus on the treatment of a specific data type, leaving a critical need for tools for integrative analysis of multiple genomic assays for locally generated or publicly available data. R / Bioconductor has historically provided standardized genomic data structures and annotations that have enjoyed widespread adoption in the cancer genomics research community. This proposal adapts R / Bioconductor to meet the increasing conceptual and computational complexity of multi-assay cancer genomic experiments. We begin by developing software containers for coordinated representation, manipulation, and transformation of heterogeneous derived data from multiple cancer genomic assays. These containers are then extended to manage very large primary data resources. To facilitate integration of local experimental results with major public cancer genomics experiment data sets and annotations, we re-package public resources and provide software and cloud-based facilities for easy and fast programmatic access from within R/Bioconductor. This greatly simplifies cancer genomic analysis tasks that otherwise require significant, error-prone individual efforts. Finally, we provide software infrastructure to enable high-throughput computation using parallel and iterative approaches. The ability to manipulate multi-assay cancer genomic experiments, to understand individual experimental results in the context of public experiments and annotations, and facilities for improved high-throughput computational performance in a well-established computing environment greatly enhances opportunities for analysis and comprehension of large multi-assay cancer genomic experiments.
