<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Recent Commit Activity</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-2f5df379a58b258e96c21c0638c20c03.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-d1b12f2568ecbe55642fee6aa00bd082.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="style.css">
</head>

<body class="nav-fixed fullcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">U24CancerProjectHub</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./overview.html"> 
<span class="menu-text">Overview</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./projects.html"> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./commit_log.html" aria-current="page"> 
<span class="menu-text">CommitLog</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Recent Commit Activity</h1>
<p class="subtitle lead">This site automatically tracks a curated list of GitHub projects funded by NIH grant <a href="https://reporter.nih.gov/project-details/10865962">1U24CA289073-01</a> from the National Cancer Institute to Levi Waldron at the CUNY Graduate School of Public Health and Health Policy</p>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="ohifviewers" class="level3">
<h3 class="anchored" data-anchor-id="ohifviewers"><a href="https://github.com/OHIF/Viewers/commit/ede7c4e3eee8db7332457ca96ed6fa8204c5cd4d"><code>OHIF/Viewers</code></a></h3>
<p><strong>Message:</strong> chore(version): Update package versions [skip ci]</p>
<p><strong>Author:</strong> ohif-bot</p>
<p><strong>Date:</strong> 2025-08-11 18:10:59 UTC</p>
<hr>
</section>
<section id="ohifviewers-1" class="level3">
<h3 class="anchored" data-anchor-id="ohifviewers-1"><a href="https://github.com/OHIF/Viewers/commit/a055cfe924b4217a080c4a26477a497da5bdd187"><code>OHIF/Viewers</code></a></h3>
<p><strong>Message:</strong> chore(version): version.json [skip ci]</p>
<p><strong>Author:</strong> ohif-bot</p>
<p><strong>Date:</strong> 2025-08-11 18:10:51 UTC</p>
<hr>
</section>
<section id="ohifviewers-2" class="level3">
<h3 class="anchored" data-anchor-id="ohifviewers-2"><a href="https://github.com/OHIF/Viewers/commit/0e83f4515c902ffa3dbab2d68ec81b93881e1768"><code>OHIF/Viewers</code></a></h3>
<p><strong>Message:</strong> fix: Update Dockerfile (#5319)</p>
<p><strong>Author:</strong> Ujjwal Kumar Baranwal</p>
<p><strong>Date:</strong> 2025-08-11 18:06:07 UTC</p>
<hr>
</section>
<section id="ohifviewers-3" class="level3">
<h3 class="anchored" data-anchor-id="ohifviewers-3"><a href="https://github.com/OHIF/Viewers/commit/51b2303b0a72cf8827968162d256a14a6a8d6d09"><code>OHIF/Viewers</code></a></h3>
<p><strong>Message:</strong> chore(version): Update package versions [skip ci]</p>
<p><strong>Author:</strong> ohif-bot</p>
<p><strong>Date:</strong> 2025-08-11 14:41:28 UTC</p>
<hr>
</section>
<section id="ohifviewers-4" class="level3">
<h3 class="anchored" data-anchor-id="ohifviewers-4"><a href="https://github.com/OHIF/Viewers/commit/078cac76bfed47a2e7a3396379d125867db81cc0"><code>OHIF/Viewers</code></a></h3>
<p><strong>Message:</strong> chore(version): version.json [skip ci]</p>
<p><strong>Author:</strong> ohif-bot</p>
<p><strong>Date:</strong> 2025-08-11 14:41:21 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone3d" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone3d"><a href="https://github.com/cornerstonejs/cornerstone3D/commit/643819927064304e4d9d8a8e8fdb8cffbd59bec2"><code>cornerstonejs/cornerstone3D</code></a></h3>
<p><strong>Message:</strong> chore: update generated version file [skip ci]</p>
<p><strong>Author:</strong> ohif-bot</p>
<p><strong>Date:</strong> 2025-08-11 14:29:17 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone3d-1" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone3d-1"><a href="https://github.com/cornerstonejs/cornerstone3D/commit/2840296f5a0fdf4a0abae0e44fef7ea77cd9d749"><code>cornerstonejs/cornerstone3D</code></a></h3>
<p><strong>Message:</strong> chore(version): Update package versions [skip ci]</p>
<p><strong>Author:</strong> ohif-bot</p>
<p><strong>Date:</strong> 2025-08-11 14:29:09 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone3d-2" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone3d-2"><a href="https://github.com/cornerstonejs/cornerstone3D/commit/501d6de14e0c1d57eaa1f8d39a612fd593564cfa"><code>cornerstonejs/cornerstone3D</code></a></h3>
<p><strong>Message:</strong> chore(version): version.json [skip ci]</p>
<p><strong>Author:</strong> ohif-bot</p>
<p><strong>Date:</strong> 2025-08-11 14:28:57 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone3d-3" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone3d-3"><a href="https://github.com/cornerstonejs/cornerstone3D/commit/9ea91402a277b50cc233382b8d1f09b5725f43ab"><code>cornerstonejs/cornerstone3D</code></a></h3>
<p><strong>Message:</strong> fix(annotation): toggle visibility off (#2262)</p>
<p><strong>Author:</strong> Devu-trenser</p>
<p><strong>Date:</strong> 2025-08-11 14:25:01 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone3d-4" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone3d-4"><a href="https://github.com/cornerstonejs/cornerstone3D/commit/670a27bd425cc0866a931ce62c9f26f6e6b1fabb"><code>cornerstonejs/cornerstone3D</code></a></h3>
<p><strong>Message:</strong> chore: update generated version file [skip ci]</p>
<p><strong>Author:</strong> ohif-bot</p>
<p><strong>Date:</strong> 2025-08-11 13:59:03 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment"><a href="https://github.com/waldronlab/MultiAssayExperiment/commit/ff721594586f1071f7fb32677c34293c985ded98"><code>waldronlab/MultiAssayExperiment</code></a></h3>
<p><strong>Message:</strong> version bump 1.35.7</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-08-08 20:18:13 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment-1"><a href="https://github.com/waldronlab/MultiAssayExperiment/commit/18ce3853af121cf27dead1df3a8c01af3c5f4f4e"><code>waldronlab/MultiAssayExperiment</code></a></h3>
<p><strong>Message:</strong> minor updates</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-08-08 20:18:07 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment-2"><a href="https://github.com/waldronlab/MultiAssayExperiment/commit/afc18063652028e9c0f05ef040346bdc6e6880a9"><code>waldronlab/MultiAssayExperiment</code></a></h3>
<p><strong>Message:</strong> update ignore files</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-08-08 20:17:56 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment-3"><a href="https://github.com/waldronlab/MultiAssayExperiment/commit/87c4d26417f210dae5a43c5f37280e375f743879"><code>waldronlab/MultiAssayExperiment</code></a></h3>
<p><strong>Message:</strong> update vignette subsetting explanation, closes #342</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-08-08 20:17:12 UTC</p>
<hr>
</section>
<section id="dcmjs-orgdcmjs" class="level3">
<h3 class="anchored" data-anchor-id="dcmjs-orgdcmjs"><a href="https://github.com/dcmjs-org/dcmjs/commit/2ca6bb1e86daf528a10c28585a02fe37bbbf1025"><code>dcmjs-org/dcmjs</code></a></h3>
<p><strong>Message:</strong> fix: Save/load large explicit encoding values (#450)</p>
<p><strong>Author:</strong> Bill Wallace</p>
<p><strong>Date:</strong> 2025-08-07 14:39:58 UTC</p>
<hr>
</section>
<section id="dcmjs-orgdcmjs-1" class="level3">
<h3 class="anchored" data-anchor-id="dcmjs-orgdcmjs-1"><a href="https://github.com/dcmjs-org/dcmjs/commit/65536e2fcaa91411b8de02981c9dfd235fc93ab8"><code>dcmjs-org/dcmjs</code></a></h3>
<p><strong>Message:</strong> build(deps): bump form-data from 3.0.1 to 3.0.4 (#446)</p>
<p><strong>Author:</strong> dependabot[bot]</p>
<p><strong>Date:</strong> 2025-08-07 01:33:35 UTC</p>
<hr>
</section>
<section id="dcmjs-orgdcmjs-2" class="level3">
<h3 class="anchored" data-anchor-id="dcmjs-orgdcmjs-2"><a href="https://github.com/dcmjs-org/dcmjs/commit/c20817e34dcb9c3114a64ac7d8fd285798ddd93a"><code>dcmjs-org/dcmjs</code></a></h3>
<p><strong>Message:</strong> feat: support DICOMs with missing meta length tag (#449)</p>
<p><strong>Author:</strong> Ofek Bashan</p>
<p><strong>Date:</strong> 2025-08-06 22:23:46 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/37df7650985db64f0e06b72728dab7909a391ce8"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> handling updated delimiter</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-28 18:48:32 UTC</p>
<hr>
</section>
<section id="shbriefomicsmlrepor" class="level3">
<h3 class="anchored" data-anchor-id="shbriefomicsmlrepor"><a href="https://github.com/shbrief/OmicsMLRepoR/commit/a3469d87af4166ea3a4295b0658aa81f0ec9c292"><code>shbrief/OmicsMLRepoR</code></a></h3>
<p><strong>Message:</strong> version bump</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-26 18:09:08 UTC</p>
<hr>
</section>
<section id="shbriefomicsmlrepor-1" class="level3">
<h3 class="anchored" data-anchor-id="shbriefomicsmlrepor-1"><a href="https://github.com/shbrief/OmicsMLRepoR/commit/bf028a8c5cac3cfecba9cc53e5331f93302eaadd"><code>shbrief/OmicsMLRepoR</code></a></h3>
<p><strong>Message:</strong> Merge pull request #50 from shbrief/feature_ontoSummarize</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-26 17:52:38 UTC</p>
<hr>
</section>
<section id="shbriefomicsmlrepor-2" class="level3">
<h3 class="anchored" data-anchor-id="shbriefomicsmlrepor-2"><a href="https://github.com/shbrief/OmicsMLRepoR/commit/fec4878db887b6f53ac2e1c84d5cec9ef4017e96"><code>shbrief/OmicsMLRepoR</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘devel’ into feature_ontoSummarize</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-26 17:52:30 UTC</p>
<hr>
</section>
<section id="shbriefomicsmlrepor-3" class="level3">
<h3 class="anchored" data-anchor-id="shbriefomicsmlrepor-3"><a href="https://github.com/shbrief/OmicsMLRepoR/commit/9f03f07211f7a9deb727ce6ca7574c0a731beb27"><code>shbrief/OmicsMLRepoR</code></a></h3>
<p><strong>Message:</strong> updated metadata</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-26 17:48:31 UTC</p>
<hr>
</section>
<section id="shbriefomicsmlrepor-4" class="level3">
<h3 class="anchored" data-anchor-id="shbriefomicsmlrepor-4"><a href="https://github.com/shbrief/OmicsMLRepoR/commit/199efbcec42e8512ef3b8d57d4a48d148eb40009"><code>shbrief/OmicsMLRepoR</code></a></h3>
<p><strong>Message:</strong> add cBioPortal data dictionary</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-26 17:45:20 UTC</p>
<hr>
</section>
<section id="waldronlabomicsmlrepodata" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabomicsmlrepodata"><a href="https://github.com/waldronlab/OmicsMLRepoData/commit/83565d766632da633be09c9d309fdc264450196d"><code>waldronlab/OmicsMLRepoData</code></a></h3>
<p><strong>Message:</strong> update cMD data dictionary</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-26 15:30:32 UTC</p>
<hr>
</section>
<section id="waldronlabomicsmlrepodata-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabomicsmlrepodata-1"><a href="https://github.com/waldronlab/OmicsMLRepoData/commit/179d8b4fca85969b00b5a4226e1da4bfd2bf32f5"><code>waldronlab/OmicsMLRepoData</code></a></h3>
<p><strong>Message:</strong> update cBioPortal data dictionary</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-26 15:29:04 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm-1" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm-1"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/ff26cc4cee4d37748f626a21c69a90aad484b624"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> update delimiter</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-25 22:53:37 UTC</p>
<hr>
</section>
<section id="waldronlabomicsmlrepodata-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabomicsmlrepodata-2"><a href="https://github.com/waldronlab/OmicsMLRepoData/commit/bdb8534f70498150bbab791b92ee51655c508ca6"><code>waldronlab/OmicsMLRepoData</code></a></h3>
<p><strong>Message:</strong> updated curations</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-25 19:38:11 UTC</p>
<hr>
</section>
<section id="waldronlabomicsmlrepodata-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabomicsmlrepodata-3"><a href="https://github.com/waldronlab/OmicsMLRepoData/commit/e7d55143350b103430c3e37f57fb071c9d3849f9"><code>waldronlab/OmicsMLRepoData</code></a></h3>
<p><strong>Message:</strong> oncotree map</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-25 19:37:22 UTC</p>
<hr>
</section>
<section id="waldronlabomicsmlrepodata-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabomicsmlrepodata-4"><a href="https://github.com/waldronlab/OmicsMLRepoData/commit/f537a479f85d257372b93cccb878b41457bc93cf"><code>waldronlab/OmicsMLRepoData</code></a></h3>
<p><strong>Message:</strong> updated cMD metadata</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-25 19:35:12 UTC</p>
<hr>
</section>
<section id="bioconductorannotationhub" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorannotationhub"><a href="https://github.com/Bioconductor/AnnotationHub/commit/11758ef932695f2b2294f0a81376b64ef7065f87"><code>Bioconductor/AnnotationHub</code></a></h3>
<p><strong>Message:</strong> Merge pull request #63 from Bioconductor/selectSome</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-07-25 12:12:30 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment-4"><a href="https://github.com/waldronlab/MultiAssayExperiment/commit/0ed7ef7ac594c12e585850c07dcb2a277899df27"><code>waldronlab/MultiAssayExperiment</code></a></h3>
<p><strong>Message:</strong> version bump 1.35.6</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-23 16:21:41 UTC</p>
<hr>
</section>
<section id="waldronlabbedbaser" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbedbaser"><a href="https://github.com/waldronlab/bedbaser/commit/54e87308eeb9e20b2d63cce7dc060862b0315b8d"><code>waldronlab/bedbaser</code></a></h3>
<p><strong>Message:</strong> Update action, use release for pkgdown site (#19)</p>
<p><strong>Author:</strong> jwokaty</p>
<p><strong>Date:</strong> 2025-07-22 14:24:42 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedtcgadata" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedtcgadata"><a href="https://github.com/waldronlab/curatedTCGAData/commit/00a83d5386ddddbf4dde7b464d6b83e575160d1f"><code>waldronlab/curatedTCGAData</code></a></h3>
<p><strong>Message:</strong> version bump 1.31.2</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 19:17:15 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedtcgadata-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedtcgadata-1"><a href="https://github.com/waldronlab/curatedTCGAData/commit/45feb0097243b5194c00c564b0349aff65336217"><code>waldronlab/curatedTCGAData</code></a></h3>
<p><strong>Message:</strong> add chunk labels to vignette</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 19:17:13 UTC</p>
<hr>
</section>
<section id="waldronlabcbioportaldata" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcbioportaldata"><a href="https://github.com/waldronlab/cBioPortalData/commit/acba625192ad8a8bfabd261132c644ed77ddea5b"><code>waldronlab/cBioPortalData</code></a></h3>
<p><strong>Message:</strong> version bump 2.21.5</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 19:12:16 UTC</p>
<hr>
</section>
<section id="waldronlabcbioportaldata-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcbioportaldata-1"><a href="https://github.com/waldronlab/cBioPortalData/commit/ad778c123b1e04bff7117ab9ccc244955ef3dfcf"><code>waldronlab/cBioPortalData</code></a></h3>
<p><strong>Message:</strong> add chunk labels to vignettes</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 19:11:37 UTC</p>
<hr>
</section>
<section id="waldronlabcbioportaldata-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcbioportaldata-2"><a href="https://github.com/waldronlab/cBioPortalData/commit/8173b25047b02c6a14b64c55f0a7e068a07d11ed"><code>waldronlab/cBioPortalData</code></a></h3>
<p><strong>Message:</strong> avoid NSE subset</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 19:11:18 UTC</p>
<hr>
</section>
<section id="waldronlabcbioportaldata-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcbioportaldata-3"><a href="https://github.com/waldronlab/cBioPortalData/commit/27ff6f3626c23d6166b42aae0df884e2003eca87"><code>waldronlab/cBioPortalData</code></a></h3>
<p><strong>Message:</strong> shorten line to 80 width limit</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 19:08:37 UTC</p>
<hr>
</section>
<section id="bioconductorgenomicfiles" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorgenomicfiles"><a href="https://github.com/Bioconductor/GenomicFiles/commit/53f71f2faab38c932692d0d0b96a31d706ceac75"><code>Bioconductor/GenomicFiles</code></a></h3>
<p><strong>Message:</strong> version bump 1.45.2</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 15:45:02 UTC</p>
<hr>
</section>
<section id="bioconductorgenomicfiles-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorgenomicfiles-1"><a href="https://github.com/Bioconductor/GenomicFiles/commit/c69bd022767809b74fe19a03231849e13a7978cf"><code>Bioconductor/GenomicFiles</code></a></h3>
<p><strong>Message:</strong> add URL and BugReports</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 15:44:56 UTC</p>
<hr>
</section>
<section id="bioconductorgenomicfiles-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorgenomicfiles-2"><a href="https://github.com/Bioconductor/GenomicFiles/commit/8ec78d2691dfdeec690739198fac37f9ce8c50f9"><code>Bioconductor/GenomicFiles</code></a></h3>
<p><strong>Message:</strong> remove whitespaces</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 15:44:45 UTC</p>
<hr>
</section>
<section id="bioconductorgenomicfiles-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorgenomicfiles-3"><a href="https://github.com/Bioconductor/GenomicFiles/commit/f532033bb03269489b2a0f753c5b5425cedacd4d"><code>Bioconductor/GenomicFiles</code></a></h3>
<p><strong>Message:</strong> update versions in DESC</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 15:43:53 UTC</p>
<hr>
</section>
<section id="bioconductorgenomicfiles-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorgenomicfiles-4"><a href="https://github.com/Bioconductor/GenomicFiles/commit/c5e381900773eaa1ad6e1cc299918a3bfbc4f168"><code>Bioconductor/GenomicFiles</code></a></h3>
<p><strong>Message:</strong> format DESCRIPTION</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-21 15:42:32 UTC</p>
<hr>
</section>
<section id="bioconductorsparsearray" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorsparsearray"><a href="https://github.com/Bioconductor/SparseArray/commit/0a2c62a32979b9bc89499621bddb5a8e8520a568"><code>Bioconductor/SparseArray</code></a></h3>
<p><strong>Message:</strong> SparseArray 1.9.1</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-07-18 05:10:47 UTC</p>
<hr>
</section>
<section id="bioconductorannotationhub-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorannotationhub-1"><a href="https://github.com/Bioconductor/AnnotationHub/commit/f3bfe3e99b22f3eb9caf3a428f5022e039580e5d"><code>Bioconductor/AnnotationHub</code></a></h3>
<p><strong>Message:</strong> use BiocBaseUtils::selectSome</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-17 19:28:50 UTC</p>
<hr>
</section>
<section id="qiicrdcmqi" class="level3">
<h3 class="anchored" data-anchor-id="qiicrdcmqi"><a href="https://github.com/QIICR/dcmqi/commit/afa4912fb05817fb8632ca0141618d52ac219510"><code>QIICR/dcmqi</code></a></h3>
<p><strong>Message:</strong> Fix DCMTK version comment (3.6.9++ not 3.6.8++)</p>
<p><strong>Author:</strong> Michael Onken</p>
<p><strong>Date:</strong> 2025-07-17 11:55:58 UTC</p>
<hr>
</section>
<section id="qiicrdcmqi-1" class="level3">
<h3 class="anchored" data-anchor-id="qiicrdcmqi-1"><a href="https://github.com/QIICR/dcmqi/commit/bd2015c3b78f7d0e9743f1cea469870da8f3c1e1"><code>QIICR/dcmqi</code></a></h3>
<p><strong>Message:</strong> Switch to windows-2022 runner for publish step.</p>
<p><strong>Author:</strong> Michael Onken</p>
<p><strong>Date:</strong> 2025-07-16 19:35:34 UTC</p>
<hr>
</section>
<section id="qiicrdcmqi-2" class="level3">
<h3 class="anchored" data-anchor-id="qiicrdcmqi-2"><a href="https://github.com/QIICR/dcmqi/commit/09e1833e24d07a9f3eeba6c07a8442a1ce39dd66"><code>QIICR/dcmqi</code></a></h3>
<p><strong>Message:</strong> Switch to official DCMTK 3.6.8++ (June 6th 2025).</p>
<p><strong>Author:</strong> Michael Onken</p>
<p><strong>Date:</strong> 2025-07-16 18:24:49 UTC</p>
<hr>
</section>
<section id="bioconductorbiocfilecache" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocfilecache"><a href="https://github.com/Bioconductor/BiocFileCache/commit/0f8c62bf901f940a155829588d94828f3b11041f"><code>Bioconductor/BiocFileCache</code></a></h3>
<p><strong>Message:</strong> Merge pull request #65 from Bioconductor/bfc-from-df</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-07-16 16:51:49 UTC</p>
<hr>
</section>
<section id="bioconductorexperimenthub" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorexperimenthub"><a href="https://github.com/Bioconductor/ExperimentHub/commit/f2a82b504abea3cf7eb9806afc5c12229a5c3e70"><code>Bioconductor/ExperimentHub</code></a></h3>
<p><strong>Message:</strong> Merge pull request #50 from Bioconductor/pkgdown_prep</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-07-16 16:51:02 UTC</p>
<hr>
</section>
<section id="bioconductorannotationhub-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorannotationhub-2"><a href="https://github.com/Bioconductor/AnnotationHub/commit/fa85eadf3f0d059fdebb4d64f7043ed9ab50d537"><code>Bioconductor/AnnotationHub</code></a></h3>
<p><strong>Message:</strong> Merge pull request #62 from Bioconductor/bioccheck_updates</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-07-16 16:48:47 UTC</p>
<hr>
</section>
<section id="waldronlabbedbaser-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbedbaser-1"><a href="https://github.com/waldronlab/bedbaser/commit/7a7e8055777f3d73f2847126de368721da6f6c38"><code>waldronlab/bedbaser</code></a></h3>
<p><strong>Message:</strong> Bump to 1.1.2</p>
<p><strong>Author:</strong> Wokaty</p>
<p><strong>Date:</strong> 2025-07-16 15:48:23 UTC</p>
<hr>
</section>
<section id="waldronlabbedbaser-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbedbaser-2"><a href="https://github.com/waldronlab/bedbaser/commit/9fe0fcf6257d6f1054357ca7ff34c72c66674990"><code>waldronlab/bedbaser</code></a></h3>
<p><strong>Message:</strong> Remove bigbed from vignettes</p>
<p><strong>Author:</strong> Wokaty</p>
<p><strong>Date:</strong> 2025-07-16 15:47:50 UTC</p>
<hr>
</section>
<section id="waldronlabbedbaser-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbedbaser-3"><a href="https://github.com/waldronlab/bedbaser/commit/8500979e64a4c0a415064f39454c33e25c1c3c84"><code>waldronlab/bedbaser</code></a></h3>
<p><strong>Message:</strong> Use release branch</p>
<p><strong>Author:</strong> Wokaty</p>
<p><strong>Date:</strong> 2025-07-16 14:21:27 UTC</p>
<hr>
</section>
<section id="seandavibiocpkgtools" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocpkgtools"><a href="https://github.com/seandavi/BiocPkgTools/commit/0a55068957d1655ee77801b691691f40c85cbdab"><code>seandavi/BiocPkgTools</code></a></h3>
<p><strong>Message:</strong> version bump 1.27.9</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-15 18:23:40 UTC</p>
<hr>
</section>
<section id="seandavibiocpkgtools-1" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocpkgtools-1"><a href="https://github.com/seandavi/BiocPkgTools/commit/0adb6e0efd50d609b67e72bb4f82192d0a9a00f9"><code>seandavi/BiocPkgTools</code></a></h3>
<p><strong>Message:</strong> update NEWS</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-15 18:22:09 UTC</p>
<hr>
</section>
<section id="seandavibiocpkgtools-2" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocpkgtools-2"><a href="https://github.com/seandavi/BiocPkgTools/commit/0218f71f62e3bb045efefc5b0724ef6efd4453bf"><code>seandavi/BiocPkgTools</code></a></h3>
<p><strong>Message:</strong> use pkgndep to generate dependency_heatmap</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-15 18:20:53 UTC</p>
<hr>
</section>
<section id="seandavibiocpkgtools-3" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocpkgtools-3"><a href="https://github.com/seandavi/BiocPkgTools/commit/590d02ee3efca4ef1ec9714d81af8377e1a69ea3"><code>seandavi/BiocPkgTools</code></a></h3>
<p><strong>Message:</strong> update formatting of template</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-15 18:20:35 UTC</p>
<hr>
</section>
<section id="seandavibiocpkgtools-4" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocpkgtools-4"><a href="https://github.com/seandavi/BiocPkgTools/commit/2d334a5b417cc398822b591b81956625b8195cc6"><code>seandavi/BiocPkgTools</code></a></h3>
<p><strong>Message:</strong> update title</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-15 18:20:01 UTC</p>
<hr>
</section>
<section id="waldronlabbedbaser-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbedbaser-4"><a href="https://github.com/waldronlab/bedbaser/commit/d8e58c8c99becf59e16a4194e724d666ced3cc94"><code>waldronlab/bedbaser</code></a></h3>
<p><strong>Message:</strong> Remove devel in action</p>
<p><strong>Author:</strong> Wokaty</p>
<p><strong>Date:</strong> 2025-07-15 14:26:38 UTC</p>
<hr>
</section>
<section id="bioconductorbiocfilecache-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocfilecache-1"><a href="https://github.com/Bioconductor/BiocFileCache/commit/9cfbd1d0e295775ffdfc123d28470587a018188f"><code>Bioconductor/BiocFileCache</code></a></h3>
<p><strong>Message:</strong> add makeBiocFileCacheFromDataFrame example</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-15 00:21:42 UTC</p>
<hr>
</section>
<section id="bioconductorexperimenthub-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorexperimenthub-1"><a href="https://github.com/Bioconductor/ExperimentHub/commit/775f62a471e55c24e597a1df0ad575221fbff6f5"><code>Bioconductor/ExperimentHub</code></a></h3>
<p><strong>Message:</strong> load namespace and attach to search list</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-14 23:56:25 UTC</p>
<hr>
</section>
<section id="bioconductorannotationhub-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorannotationhub-3"><a href="https://github.com/Bioconductor/AnnotationHub/commit/ad2ed509d2a4651c83ba1c87e8fb33022a732297"><code>Bioconductor/AnnotationHub</code></a></h3>
<p><strong>Message:</strong> expand example in convertHub doc</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-14 23:37:51 UTC</p>
<hr>
</section>
<section id="dcmjs-orgdcmjs-3" class="level3">
<h3 class="anchored" data-anchor-id="dcmjs-orgdcmjs-3"><a href="https://github.com/dcmjs-org/dcmjs/commit/f9dac44ed6676751847d78ee2ce4ae603afc0910"><code>dcmjs-org/dcmjs</code></a></h3>
<p><strong>Message:</strong> fix(data): Use an array of data for BufferStream (#444)</p>
<p><strong>Author:</strong> Bill Wallace</p>
<p><strong>Date:</strong> 2025-07-14 18:04:19 UTC</p>
<hr>
</section>
<section id="bioconductorraggedexperiment" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorraggedexperiment"><a href="https://github.com/Bioconductor/RaggedExperiment/commit/4cb9ce7518ddda891309a239ba5af84b3bb64055"><code>Bioconductor/RaggedExperiment</code></a></h3>
<p><strong>Message:</strong> version bump 1.33.7</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-10 18:19:28 UTC</p>
<hr>
</section>
<section id="bioconductorraggedexperiment-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorraggedexperiment-1"><a href="https://github.com/Bioconductor/RaggedExperiment/commit/956b071e538af4d55d20d9b11c7586ee95abd2a2"><code>Bioconductor/RaggedExperiment</code></a></h3>
<p><strong>Message:</strong> roxygen package Rd update</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-10 18:19:22 UTC</p>
<hr>
</section>
<section id="bioconductorraggedexperiment-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorraggedexperiment-2"><a href="https://github.com/Bioconductor/RaggedExperiment/commit/1a997f8542f9f8dde0c5a671b71bf69081a11a71"><code>Bioconductor/RaggedExperiment</code></a></h3>
<p><strong>Message:</strong> move chunk options to code chunk body</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-10 18:19:09 UTC</p>
<hr>
</section>
<section id="bioconductorraggedexperiment-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorraggedexperiment-3"><a href="https://github.com/Bioconductor/RaggedExperiment/commit/ff60b8f4a54f73b7d7853632144fca0fa94aad9f"><code>Bioconductor/RaggedExperiment</code></a></h3>
<p><strong>Message:</strong> version bump 1.33.6</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-10 17:55:39 UTC</p>
<hr>
</section>
<section id="bioconductorraggedexperiment-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorraggedexperiment-4"><a href="https://github.com/Bioconductor/RaggedExperiment/commit/ad5aeb963a9264a0750dafd366cac4807a06e038"><code>Bioconductor/RaggedExperiment</code></a></h3>
<p><strong>Message:</strong> remove long link</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-10 17:55:27 UTC</p>
<hr>
</section>
<section id="dcmjs-orgdcmjs-4" class="level3">
<h3 class="anchored" data-anchor-id="dcmjs-orgdcmjs-4"><a href="https://github.com/dcmjs-org/dcmjs/commit/315a829d89e55373f725992ceff967dea9519240"><code>dcmjs-org/dcmjs</code></a></h3>
<p><strong>Message:</strong> feat(build): Create minified UMD build (#443)</p>
<p><strong>Author:</strong> Harm van Zeeland</p>
<p><strong>Date:</strong> 2025-07-10 12:36:59 UTC</p>
<hr>
</section>
<section id="bd4everyoneu24project" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyoneu24project"><a href="https://github.com/bd4everyone/u24Project/commit/b489979e1da435cefcaa6807838b5ad8e08cf5d8"><code>bd4everyone/u24Project</code></a></h3>
<p><strong>Message:</strong> Create jekyll-gh-pages.yml</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-07-08 17:37:10 UTC</p>
<hr>
</section>
<section id="bd4everyoneu24project-1" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyoneu24project-1"><a href="https://github.com/bd4everyone/u24Project/commit/86bc22d4fed90774ae665f271a2ed15403020d09"><code>bd4everyone/u24Project</code></a></h3>
<p><strong>Message:</strong> Delete docs directory</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-07-08 17:28:57 UTC</p>
<hr>
</section>
<section id="bd4everyoneu24project-2" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyoneu24project-2"><a href="https://github.com/bd4everyone/u24Project/commit/1fce18551611f58906fe5110ad18f9d04ee26735"><code>bd4everyone/u24Project</code></a></h3>
<p><strong>Message:</strong> Create empty</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-07-08 17:28:33 UTC</p>
<hr>
</section>
<section id="bd4everyoneu24project-3" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyoneu24project-3"><a href="https://github.com/bd4everyone/u24Project/commit/dbba4e902b58cd698167eaecb5cc6ee7c2cafbf5"><code>bd4everyone/u24Project</code></a></h3>
<p><strong>Message:</strong> Update index.md</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-07-08 17:07:17 UTC</p>
<hr>
</section>
<section id="bd4everyoneu24project-4" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyoneu24project-4"><a href="https://github.com/bd4everyone/u24Project/commit/4c1e81f9fba093a83dbbbbf40e1fd95789b921b6"><code>bd4everyone/u24Project</code></a></h3>
<p><strong>Message:</strong> Update index.md</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-07-08 17:06:54 UTC</p>
<hr>
</section>
<section id="waldronlabxeniumio" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabxeniumio"><a href="https://github.com/waldronlab/XeniumIO/commit/94e988c170a91a576ddb9c32ef1d3d302019539d"><code>waldronlab/XeniumIO</code></a></h3>
<p><strong>Message:</strong> version bump 1.1.3</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-03 20:58:11 UTC</p>
<hr>
</section>
<section id="waldronlabxeniumio-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabxeniumio-1"><a href="https://github.com/waldronlab/XeniumIO/commit/a7516775c1b54f9d0239de82f54171855763fb17"><code>waldronlab/XeniumIO</code></a></h3>
<p><strong>Message:</strong> remove rtype arg from bfcdownload</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-03 20:58:09 UTC</p>
<hr>
</section>
<section id="waldronlabcbioportaldata-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcbioportaldata-4"><a href="https://github.com/waldronlab/cBioPortalData/commit/28ec34054b440f2e3d51e0bfda877917b67d25e8"><code>waldronlab/cBioPortalData</code></a></h3>
<p><strong>Message:</strong> version bump 2.21.4</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 20:34:20 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayworkshop" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayworkshop"><a href="https://github.com/waldronlab/MultiAssayWorkshop/commit/0deb00bac8c04a883156bbc6545f15894e85cc34"><code>waldronlab/MultiAssayWorkshop</code></a></h3>
<p><strong>Message:</strong> add concurrency limits</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 17:44:25 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayworkshop-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayworkshop-1"><a href="https://github.com/waldronlab/MultiAssayWorkshop/commit/cf8ca4150a07e94e64d16564e0807f313804376f"><code>waldronlab/MultiAssayWorkshop</code></a></h3>
<p><strong>Message:</strong> use multiline sep</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 16:18:27 UTC</p>
<hr>
</section>
<section id="waldronlabtcgautils" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtcgautils"><a href="https://github.com/waldronlab/TCGAutils/commit/bafe4cea87ec74a66906299649b46814666bb0c3"><code>waldronlab/TCGAutils</code></a></h3>
<p><strong>Message:</strong> bump version 1.29.5</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 16:15:08 UTC</p>
<hr>
</section>
<section id="waldronlabtcgautils-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtcgautils-1"><a href="https://github.com/waldronlab/TCGAutils/commit/c507a4cbc51ca15af17c29df28614c5476cb7e63"><code>waldronlab/TCGAutils</code></a></h3>
<p><strong>Message:</strong> import genome and genome&lt;- from Seqinfo</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 16:10:13 UTC</p>
<hr>
</section>
<section id="waldronlabtcgautils-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtcgautils-2"><a href="https://github.com/waldronlab/TCGAutils/commit/372581c03a6f3065c2cf9d20bf39f907c806c129"><code>waldronlab/TCGAutils</code></a></h3>
<p><strong>Message:</strong> add GenomeInfoDb to vignette</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 16:09:54 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayworkshop-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayworkshop-2"><a href="https://github.com/waldronlab/MultiAssayWorkshop/commit/6e5aa99202c69862c95090a4a5578a608b145c7e"><code>waldronlab/MultiAssayWorkshop</code></a></h3>
<p><strong>Message:</strong> version bump 3.21.4</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 16:01:21 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayworkshop-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayworkshop-3"><a href="https://github.com/waldronlab/MultiAssayWorkshop/commit/e2e928155a2d6d222360c4b3f6563bded4774bb4"><code>waldronlab/MultiAssayWorkshop</code></a></h3>
<p><strong>Message:</strong> move Date close to Version</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 16:00:37 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayworkshop-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayworkshop-4"><a href="https://github.com/waldronlab/MultiAssayWorkshop/commit/b863ab856356782abf8e3a617589ea414545bf02"><code>waldronlab/MultiAssayWorkshop</code></a></h3>
<p><strong>Message:</strong> rename longFormat to longForm</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 15:52:39 UTC</p>
<hr>
</section>
<section id="waldronlabtcgautils-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtcgautils-3"><a href="https://github.com/waldronlab/TCGAutils/commit/260183dc4bdce6cd0fc0fa401f22ce448b2436c7"><code>waldronlab/TCGAutils</code></a></h3>
<p><strong>Message:</strong> version bump 1.29.4</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 15:24:46 UTC</p>
<hr>
</section>
<section id="waldronlabtcgautils-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtcgautils-4"><a href="https://github.com/waldronlab/TCGAutils/commit/40a570c8fee209f9e4831e01cb2cd000bca2f718"><code>waldronlab/TCGAutils</code></a></h3>
<p><strong>Message:</strong> add package argument to lifeCycle</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-07-02 15:24:44 UTC</p>
<hr>
</section>
<section id="bioconductorannotationhub-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorannotationhub-4"><a href="https://github.com/Bioconductor/AnnotationHub/commit/21cf78944d2680027df20094bbdf794bd423e7f4"><code>Bioconductor/AnnotationHub</code></a></h3>
<p><strong>Message:</strong> AnnotationHub 3.99.6: Add Seqinfo to Suggests</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-06-20 23:18:03 UTC</p>
<hr>
</section>
<section id="nih-cfdeprogramets" class="level3">
<h3 class="anchored" data-anchor-id="nih-cfdeprogramets"><a href="https://github.com/nih-cfde/programets/commit/be7891207554f90babe48ab3542ec1a3bc4a0c0a"><code>nih-cfde/programets</code></a></h3>
<p><strong>Message:</strong> collapse, if multiple categores/descriptions</p>
<p><strong>Author:</strong> David Mayer</p>
<p><strong>Date:</strong> 2025-06-20 15:13:08 UTC</p>
<hr>
</section>
<section id="bioconductorsparsearray-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorsparsearray-1"><a href="https://github.com/Bioconductor/SparseArray/commit/b558d0d792275cbf27571d800a3a1d8de370a25f"><code>Bioconductor/SparseArray</code></a></h3>
<p><strong>Message:</strong> add sticker (by Charlotte) to README.md</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-06-18 16:19:07 UTC</p>
<hr>
</section>
<section id="bioconductorbiocparallel" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocparallel"><a href="https://github.com/Bioconductor/BiocParallel/commit/abe462ceb59acb88b9e4d0365df28295e14f84a6"><code>Bioconductor/BiocParallel</code></a></h3>
<p><strong>Message:</strong> Enable bpworkers&lt;- calls on SerialParam (no-op) with user message</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-06-18 14:56:30 UTC</p>
<hr>
</section>
<section id="aim-harvardpyradiomics" class="level3">
<h3 class="anchored" data-anchor-id="aim-harvardpyradiomics"><a href="https://github.com/AIM-Harvard/pyradiomics/commit/8ed579383b44806651c463d5e691f3b2b57522ab"><code>AIM-Harvard/pyradiomics</code></a></h3>
<p><strong>Message:</strong> Build(deps): Bump the actions group with 2 updates</p>
<p><strong>Author:</strong> dependabot[bot]</p>
<p><strong>Date:</strong> 2025-06-16 23:00:55 UTC</p>
<hr>
</section>
<section id="waldronlabsinglecellmultimodal" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabsinglecellmultimodal"><a href="https://github.com/waldronlab/SingleCellMultiModal/commit/a2ec4f3716a39654bfc132425ca011d519ecaf10"><code>waldronlab/SingleCellMultiModal</code></a></h3>
<p><strong>Message:</strong> version bump 1.21.3</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-06-12 16:44:59 UTC</p>
<hr>
</section>
<section id="waldronlabsinglecellmultimodal-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabsinglecellmultimodal-1"><a href="https://github.com/waldronlab/SingleCellMultiModal/commit/1e8f0a6965546c20b45e6f9b630a6a636ea981c9"><code>waldronlab/SingleCellMultiModal</code></a></h3>
<p><strong>Message:</strong> update vignette authors</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-06-12 16:44:54 UTC</p>
<hr>
</section>
<section id="bioconductorbiocparallel-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocparallel-1"><a href="https://github.com/Bioconductor/BiocParallel/commit/f72fed091af1d15cfe2ce6aca6165d26a263eacb"><code>Bioconductor/BiocParallel</code></a></h3>
<p><strong>Message:</strong> Transfer maintainership to Jiefei Wang</p>
<p><strong>Author:</strong> Jiefei Wang</p>
<p><strong>Date:</strong> 2025-06-12 02:39:17 UTC</p>
<hr>
</section>
<section id="bioconductorbiocparallel-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocparallel-2"><a href="https://github.com/Bioconductor/BiocParallel/commit/7c2254992a867bbacd26b46f94f0b1c2213f6e6c"><code>Bioconductor/BiocParallel</code></a></h3>
<p><strong>Message:</strong> version bump 1.43.3</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-28 18:00:44 UTC</p>
<hr>
</section>
<section id="bioconductorbiocparallel-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocparallel-3"><a href="https://github.com/Bioconductor/BiocParallel/commit/b3aa72c8a147578cc6934108fb7d293bc6e1608d"><code>Bioconductor/BiocParallel</code></a></h3>
<p><strong>Message:</strong> convert Rnw vignettes to Rmd (#278)</p>
<p><strong>Author:</strong> Marcel Ramos Pérez</p>
<p><strong>Date:</strong> 2025-05-28 18:00:02 UTC</p>
<hr>
</section>
<section id="bioconductors4arrays" class="level3">
<h3 class="anchored" data-anchor-id="bioconductors4arrays"><a href="https://github.com/Bioconductor/S4Arrays/commit/da54bbd15596f255cae532d352038178324d4b09"><code>Bioconductor/S4Arrays</code></a></h3>
<p><strong>Message:</strong> S4Arrays 1.9.1</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-05-27 20:15:38 UTC</p>
<hr>
</section>
<section id="nih-cfdeprogramets-1" class="level3">
<h3 class="anchored" data-anchor-id="nih-cfdeprogramets-1"><a href="https://github.com/nih-cfde/programets/commit/9c851a5ca2f666269291a844bd5cc3f31d7f5a1e"><code>nih-cfde/programets</code></a></h3>
<p><strong>Message:</strong> Merge pull request #14 from seandavi/5-collect-grant-details-for-all-cfde-projects</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-05-25 11:32:19 UTC</p>
<hr>
</section>
<section id="nih-cfdeprogramets-2" class="level3">
<h3 class="anchored" data-anchor-id="nih-cfdeprogramets-2"><a href="https://github.com/nih-cfde/programets/commit/60104de9b76f2410ee26785c39448b3148d0bc2f"><code>nih-cfde/programets</code></a></h3>
<p><strong>Message:</strong> add tryCatch for api errors</p>
<p><strong>Author:</strong> David Mayer</p>
<p><strong>Date:</strong> 2025-05-23 13:48:09 UTC</p>
<hr>
</section>
<section id="nih-cfdeprogramets-3" class="level3">
<h3 class="anchored" data-anchor-id="nih-cfdeprogramets-3"><a href="https://github.com/nih-cfde/programets/commit/c93ad0632626f24a65fdfceb1c098597c72d2d2c"><code>nih-cfde/programets</code></a></h3>
<p><strong>Message:</strong> document additional project infomation returns</p>
<p><strong>Author:</strong> David Mayer</p>
<p><strong>Date:</strong> 2025-05-23 13:41:45 UTC</p>
<hr>
</section>
<section id="nih-cfdeprogramets-4" class="level3">
<h3 class="anchored" data-anchor-id="nih-cfdeprogramets-4"><a href="https://github.com/nih-cfde/programets/commit/cf70870703d26996a9f0cf621cd3be680cf87010"><code>nih-cfde/programets</code></a></h3>
<p><strong>Message:</strong> Depend on R &gt;= 4.1.0 for new pipe</p>
<p><strong>Author:</strong> David Mayer</p>
<p><strong>Date:</strong> 2025-05-22 23:07:44 UTC</p>
<hr>
</section>
<section id="hpageshovernethelp" class="level3">
<h3 class="anchored" data-anchor-id="hpageshovernethelp"><a href="https://github.com/hpages/hovernethelp/commit/d75fd63afa44792259782190ff8653fb1e824f4a"><code>hpages/hovernethelp</code></a></h3>
<p><strong>Message:</strong> update minicluster/JOURNAL_OF_RUNS.txt</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-05-22 06:19:09 UTC</p>
<hr>
</section>
<section id="aim-harvardpyradiomics-1" class="level3">
<h3 class="anchored" data-anchor-id="aim-harvardpyradiomics-1"><a href="https://github.com/AIM-Harvard/pyradiomics/commit/9b1e030bff5241bdf9db1adb796183bb804eec25"><code>AIM-Harvard/pyradiomics</code></a></h3>
<p><strong>Message:</strong> Build(deps): Bump codecov/codecov-action in the actions group</p>
<p><strong>Author:</strong> dependabot[bot]</p>
<p><strong>Date:</strong> 2025-05-19 22:23:06 UTC</p>
<hr>
</section>
<section id="aim-harvardpyradiomics-2" class="level3">
<h3 class="anchored" data-anchor-id="aim-harvardpyradiomics-2"><a href="https://github.com/AIM-Harvard/pyradiomics/commit/7a5fea79ebea06670f547ec39ed9f81813e5a5b0"><code>AIM-Harvard/pyradiomics</code></a></h3>
<p><strong>Message:</strong> Refactor code to improve readability and consistency.</p>
<p><strong>Author:</strong> Hans Johnson</p>
<p><strong>Date:</strong> 2025-05-19 17:32:54 UTC</p>
<hr>
</section>
<section id="aim-harvardpyradiomics-3" class="level3">
<h3 class="anchored" data-anchor-id="aim-harvardpyradiomics-3"><a href="https://github.com/AIM-Harvard/pyradiomics/commit/bca2110bd62aea83abff264882a3d83d939c36d2"><code>AIM-Harvard/pyradiomics</code></a></h3>
<p><strong>Message:</strong> Refactor logging to use <code>logger.exception</code> for better tracebacks</p>
<p><strong>Author:</strong> Hans Johnson</p>
<p><strong>Date:</strong> 2025-05-19 17:25:21 UTC</p>
<hr>
</section>
<section id="aim-harvardpyradiomics-4" class="level3">
<h3 class="anchored" data-anchor-id="aim-harvardpyradiomics-4"><a href="https://github.com/AIM-Harvard/pyradiomics/commit/6e600e94c2af6fbf65a65594cbc8919a08d07e64"><code>AIM-Harvard/pyradiomics</code></a></h3>
<p><strong>Message:</strong> STYLE: G004: Refactor logging messages for consistency and clarity</p>
<p><strong>Author:</strong> Hans Johnson</p>
<p><strong>Date:</strong> 2025-05-19 17:20:11 UTC</p>
<hr>
</section>
<section id="bioconductorbiocparallel-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocparallel-4"><a href="https://github.com/Bioconductor/BiocParallel/commit/9e2db5de5f145509e494dd7668aefb2b82e6b21f"><code>Bioconductor/BiocParallel</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘Jiefei-Wang-force_load’ into devel</p>
<p><strong>Author:</strong> Martin Morgan</p>
<p><strong>Date:</strong> 2025-05-16 17:35:30 UTC</p>
<hr>
</section>
<section id="shbriefgenomicsupersignature" class="level3">
<h3 class="anchored" data-anchor-id="shbriefgenomicsupersignature"><a href="https://github.com/shbrief/GenomicSuperSignature/commit/5d18ff375fef36baf8ec7b2bfb383b7e0f946800"><code>shbrief/GenomicSuperSignature</code></a></h3>
<p><strong>Message:</strong> fix conflicting versions</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-05-16 03:16:21 UTC</p>
<hr>
</section>
<section id="bioconductorexperimenthub-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorexperimenthub-2"><a href="https://github.com/Bioconductor/ExperimentHub/commit/a616df89683e3112ce41c09cf2178687de799c51"><code>Bioconductor/ExperimentHub</code></a></h3>
<p><strong>Message:</strong> move png to man/figures</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-15 21:24:25 UTC</p>
<hr>
</section>
<section id="bioconductorexperimenthub-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorexperimenthub-3"><a href="https://github.com/Bioconductor/ExperimentHub/commit/bda9031660f984b80b6830eb15e41c614a89226f"><code>Bioconductor/ExperimentHub</code></a></h3>
<p><strong>Message:</strong> run runnable examples</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-15 21:24:10 UTC</p>
<hr>
</section>
<section id="bioconductorexperimenthub-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorexperimenthub-4"><a href="https://github.com/Bioconductor/ExperimentHub/commit/95348da04ab1a536e9dc05647477ecffbe04eb19"><code>Bioconductor/ExperimentHub</code></a></h3>
<p><strong>Message:</strong> use requireNamespace instead of require</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-15 21:23:59 UTC</p>
<hr>
</section>
<section id="bioconductorh5mread" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorh5mread"><a href="https://github.com/Bioconductor/h5mread/commit/52c036adbf717f3221bbb12ed757d674426d8d52"><code>Bioconductor/h5mread</code></a></h3>
<p><strong>Message:</strong> h5mread 1.1.1: Update BugReports field</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-05-15 18:48:59 UTC</p>
<hr>
</section>
<section id="bioconductorbiocfilecache-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocfilecache-2"><a href="https://github.com/Bioconductor/BiocFileCache/commit/e88fb8e4beb200304c4f72f2e07cbe8da6ad49a8"><code>Bioconductor/BiocFileCache</code></a></h3>
<p><strong>Message:</strong> Update Documentation for new options</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-05-15 13:35:08 UTC</p>
<hr>
</section>
<section id="shbriefgenomicsupersignature-1" class="level3">
<h3 class="anchored" data-anchor-id="shbriefgenomicsupersignature-1"><a href="https://github.com/shbrief/GenomicSuperSignature/commit/26ecc55e05c8896b396c0031215bb2d7c2a8f4b8"><code>shbrief/GenomicSuperSignature</code></a></h3>
<p><strong>Message:</strong> Merge pull request #29 from shbrief/RELEASE_3_21</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-05-15 01:33:18 UTC</p>
<hr>
</section>
<section id="bioconductorbiocfilecache-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocfilecache-3"><a href="https://github.com/Bioconductor/BiocFileCache/commit/5c2bda4803740d99a98cd0f14fde664c1f736046"><code>Bioconductor/BiocFileCache</code></a></h3>
<p><strong>Message:</strong> Merge pull request #64 from lshep/proxyBug</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-05-14 13:48:58 UTC</p>
<hr>
</section>
<section id="bioconductorbiocfilecache-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiocfilecache-4"><a href="https://github.com/Bioconductor/BiocFileCache/commit/e3747242089349cd2ff1a6dcda6697513dabf478"><code>Bioconductor/BiocFileCache</code></a></h3>
<p><strong>Message:</strong> Fix version</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-05-14 13:48:29 UTC</p>
<hr>
</section>
<section id="waldronlaboncokbdata" class="level3">
<h3 class="anchored" data-anchor-id="waldronlaboncokbdata"><a href="https://github.com/waldronlab/oncoKBData/commit/d317767bd54a07d6e6ae463f6f6630b04062232a"><code>waldronlab/oncoKBData</code></a></h3>
<p><strong>Message:</strong> version bump 0.99.7</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-09 21:26:32 UTC</p>
<hr>
</section>
<section id="waldronlaboncokbdata-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlaboncokbdata-1"><a href="https://github.com/waldronlab/oncoKBData/commit/10e2c35b456e8746593698b68aa8eb6c3ca6ec17"><code>waldronlab/oncoKBData</code></a></h3>
<p><strong>Message:</strong> import DataFrame constructor</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-09 21:26:28 UTC</p>
<hr>
</section>
<section id="waldronlaboncokbdata-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlaboncokbdata-2"><a href="https://github.com/waldronlab/oncoKBData/commit/a2782a49f5d507999097f26780e0c540f649a1eb"><code>waldronlab/oncoKBData</code></a></h3>
<p><strong>Message:</strong> version bump 0.99.6</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-09 21:16:21 UTC</p>
<hr>
</section>
<section id="waldronlaboncokbdata-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlaboncokbdata-3"><a href="https://github.com/waldronlab/oncoKBData/commit/dd2f745e21fc1324acf70061c05ebad603533668"><code>waldronlab/oncoKBData</code></a></h3>
<p><strong>Message:</strong> update RoxygenNote</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-09 21:16:16 UTC</p>
<hr>
</section>
<section id="waldronlaboncokbdata-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlaboncokbdata-4"><a href="https://github.com/waldronlab/oncoKBData/commit/fd048cfd3678d8aa7ea90fd7a00dfaa6ff7b5c84"><code>waldronlab/oncoKBData</code></a></h3>
<p><strong>Message:</strong> import from S4Vectors</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-09 21:16:08 UTC</p>
<hr>
</section>
<section id="waldronlabhgncrest" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgncrest"><a href="https://github.com/waldronlab/HGNCREST/commit/17d1839656e1ef70ad5cf18a86eaf826d4a8275c"><code>waldronlab/HGNCREST</code></a></h3>
<p><strong>Message:</strong> version bump 0.99.6</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-09 16:02:53 UTC</p>
<hr>
</section>
<section id="waldronlabhgncrest-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgncrest-1"><a href="https://github.com/waldronlab/HGNCREST/commit/dced6debc918241bca2d8acf1315cf4a8a64d357"><code>waldronlab/HGNCREST</code></a></h3>
<p><strong>Message:</strong> add .Rbuildignore and update .gitignore</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-09 16:00:09 UTC</p>
<hr>
</section>
<section id="waldronlabhgncrest-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgncrest-2"><a href="https://github.com/waldronlab/HGNCREST/commit/eafe27812139077f26deec110bbca289c0c2a2e9"><code>waldronlab/HGNCREST</code></a></h3>
<p><strong>Message:</strong> add biocViews to DESCRIPTION</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-09 15:58:59 UTC</p>
<hr>
</section>
<section id="waldronlabtenxio" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtenxio"><a href="https://github.com/waldronlab/TENxIO/commit/2f684b16bb8e382f837746658fbc9f4d7fd82e5c"><code>waldronlab/TENxIO</code></a></h3>
<p><strong>Message:</strong> version bump 1.11.4</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-08 18:22:10 UTC</p>
<hr>
</section>
<section id="waldronlabgseabenchmarker" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabgseabenchmarker"><a href="https://github.com/waldronlab/GSEABenchmarkeR/commit/c520218ae428083ea2dc2f8552028372dea29fb0"><code>waldronlab/GSEABenchmarkeR</code></a></h3>
<p><strong>Message:</strong> version bump 1.29.1</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-08 16:44:31 UTC</p>
<hr>
</section>
<section id="shbriefgenomicsupersignature-2" class="level3">
<h3 class="anchored" data-anchor-id="shbriefgenomicsupersignature-2"><a href="https://github.com/shbrief/GenomicSuperSignature/commit/08fd55fc2ba1f088f9fa61d00008b55986cfd5d3"><code>shbrief/GenomicSuperSignature</code></a></h3>
<p><strong>Message:</strong> Version bump to trigger rebuilding/propagation of Windows and Mac binaries</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-05-08 16:28:02 UTC</p>
<hr>
</section>
<section id="waldronlabcnvranger" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcnvranger"><a href="https://github.com/waldronlab/CNVRanger/commit/82ec0eab97e532e907a535d4d8540308477883ff"><code>waldronlab/CNVRanger</code></a></h3>
<p><strong>Message:</strong> version bump 1.25.4</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-08 16:17:16 UTC</p>
<hr>
</section>
<section id="waldronlabcnvranger-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcnvranger-1"><a href="https://github.com/waldronlab/CNVRanger/commit/5bbe13fa14bc9d7c15f7567c511329ce8533e418"><code>waldronlab/CNVRanger</code></a></h3>
<p><strong>Message:</strong> fix references</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-08 16:17:11 UTC</p>
<hr>
</section>
<section id="waldronlabcnvranger-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcnvranger-2"><a href="https://github.com/waldronlab/CNVRanger/commit/175c2d1f0c6563765f5be0be74b0b019db110b6d"><code>waldronlab/CNVRanger</code></a></h3>
<p><strong>Message:</strong> version bump 1.25.3</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-08 15:04:54 UTC</p>
<hr>
</section>
<section id="drighellispatialexperiment" class="level3">
<h3 class="anchored" data-anchor-id="drighellispatialexperiment"><a href="https://github.com/drighelli/SpatialExperiment/commit/0b2953777e17bd031b693315102787afa0f252c8"><code>drighelli/SpatialExperiment</code></a></h3>
<p><strong>Message:</strong> Merge pull request #167 from lmweber/devel</p>
<p><strong>Author:</strong> Lukas Weber</p>
<p><strong>Date:</strong> 2025-05-08 14:03:51 UTC</p>
<hr>
</section>
<section id="waldronlabcnvranger-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcnvranger-3"><a href="https://github.com/waldronlab/CNVRanger/commit/ce91e6ad832586135c12dca258f0b47f2b88b974"><code>waldronlab/CNVRanger</code></a></h3>
<p><strong>Message:</strong> use bibliography</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-07 23:56:38 UTC</p>
<hr>
</section>
<section id="waldronlabcnvranger-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcnvranger-4"><a href="https://github.com/waldronlab/CNVRanger/commit/ef20c16453b965e636f26951617d346abe8cbd5e"><code>waldronlab/CNVRanger</code></a></h3>
<p><strong>Message:</strong> move png files to man/figures</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-07 22:46:10 UTC</p>
<hr>
</section>
<section id="drighellispatialexperiment-1" class="level3">
<h3 class="anchored" data-anchor-id="drighellispatialexperiment-1"><a href="https://github.com/drighelli/SpatialExperiment/commit/a8fc00f931fce6b0e31ae58fede5f4488ff4e8e5"><code>drighelli/SpatialExperiment</code></a></h3>
<p><strong>Message:</strong> version</p>
<p><strong>Author:</strong> Lukas Weber</p>
<p><strong>Date:</strong> 2025-05-07 22:28:21 UTC</p>
<hr>
</section>
<section id="drighellispatialexperiment-2" class="level3">
<h3 class="anchored" data-anchor-id="drighellispatialexperiment-2"><a href="https://github.com/drighelli/SpatialExperiment/commit/8fabc05270c66d850694d5b3b6b58b284e54eca1"><code>drighelli/SpatialExperiment</code></a></h3>
<p><strong>Message:</strong> update GHA</p>
<p><strong>Author:</strong> Lukas Weber</p>
<p><strong>Date:</strong> 2025-05-07 22:27:24 UTC</p>
<hr>
</section>
<section id="drighellispatialexperiment-3" class="level3">
<h3 class="anchored" data-anchor-id="drighellispatialexperiment-3"><a href="https://github.com/drighelli/SpatialExperiment/commit/3fa5af24083b30b0845a63658913451e14a44da4"><code>drighelli/SpatialExperiment</code></a></h3>
<p><strong>Message:</strong> update email</p>
<p><strong>Author:</strong> Lukas Weber</p>
<p><strong>Date:</strong> 2025-05-07 22:26:51 UTC</p>
<hr>
</section>
<section id="drighellispatialexperiment-4" class="level3">
<h3 class="anchored" data-anchor-id="drighellispatialexperiment-4"><a href="https://github.com/drighelli/SpatialExperiment/commit/1e55cde8eb2ce9a73bac157b318e2aaa0c18aa66"><code>drighelli/SpatialExperiment</code></a></h3>
<p><strong>Message:</strong> Merge remote-tracking branch ‘upstream/devel’ into devel</p>
<p><strong>Author:</strong> Lukas Weber</p>
<p><strong>Date:</strong> 2025-05-07 22:22:38 UTC</p>
<hr>
</section>
<section id="waldronlabgseabenchmarker-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabgseabenchmarker-1"><a href="https://github.com/waldronlab/GSEABenchmarkeR/commit/2cb5fb56ac7c24238b6005a3f309a8643268012d"><code>waldronlab/GSEABenchmarkeR</code></a></h3>
<p><strong>Message:</strong> Merge pull request #17 from waldronlab/build_site_gha</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2025-05-07 22:12:33 UTC</p>
<hr>
</section>
<section id="waldronlabgseabenchmarker-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabgseabenchmarker-2"><a href="https://github.com/waldronlab/GSEABenchmarkeR/commit/44804cd1979991a6f9675011d4a49a0c6a76c572"><code>waldronlab/GSEABenchmarkeR</code></a></h3>
<p><strong>Message:</strong> resolve build warning and note</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-07 19:39:59 UTC</p>
<hr>
</section>
<section id="waldronlabgseabenchmarker-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabgseabenchmarker-3"><a href="https://github.com/waldronlab/GSEABenchmarkeR/commit/432effa9398ad4d852756375f794282685f3463c"><code>waldronlab/GSEABenchmarkeR</code></a></h3>
<p><strong>Message:</strong> use Authors@R</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-07 19:35:46 UTC</p>
<hr>
</section>
<section id="waldronlabgseabenchmarker-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabgseabenchmarker-4"><a href="https://github.com/waldronlab/GSEABenchmarkeR/commit/0e46f4b5b1f13ace6487a84a57b8b66f822c0b25"><code>waldronlab/GSEABenchmarkeR</code></a></h3>
<p><strong>Message:</strong> add _pkgdown.yml and .github/workflows/pr_check.yml GHA</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-07 19:32:24 UTC</p>
<hr>
</section>
<section id="waldronlabhgncrest-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgncrest-3"><a href="https://github.com/waldronlab/HGNCREST/commit/025d99c9c1ce67c2a226ab73c4c1de32497b67a2"><code>waldronlab/HGNCREST</code></a></h3>
<p><strong>Message:</strong> version bump 0.99.5</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-07 15:57:43 UTC</p>
<hr>
</section>
<section id="waldronlabhgncrest-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgncrest-4"><a href="https://github.com/waldronlab/HGNCREST/commit/f28c6e64a585a1cca4e754199aca74f7bf7103d1"><code>waldronlab/HGNCREST</code></a></h3>
<p><strong>Message:</strong> .gitignore html</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-07 15:57:38 UTC</p>
<hr>
</section>
<section id="waldronlabvisiumio" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabvisiumio"><a href="https://github.com/waldronlab/VisiumIO/commit/0ffa1d44442a5af032702f9514ed5ae161082e5e"><code>waldronlab/VisiumIO</code></a></h3>
<p><strong>Message:</strong> version bump 1.5.6</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-07 15:07:34 UTC</p>
<hr>
</section>
<section id="waldronlabvisiumio-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabvisiumio-1"><a href="https://github.com/waldronlab/VisiumIO/commit/d856e70231953d64ac4a66b35fc267580f155d79"><code>waldronlab/VisiumIO</code></a></h3>
<p><strong>Message:</strong> update .Rbuildignore</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-07 15:07:26 UTC</p>
<hr>
</section>
<section id="hpageshovernethelp-1" class="level3">
<h3 class="anchored" data-anchor-id="hpageshovernethelp-1"><a href="https://github.com/hpages/hovernethelp/commit/2af327c7268b0277848a4873bce4349af9ca8fde"><code>hpages/hovernethelp</code></a></h3>
<p><strong>Message:</strong> update minicluster/JOURNAL_OF_RUNS.txt</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-05-06 23:28:36 UTC</p>
<hr>
</section>
<section id="waldronlabtenxio-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtenxio-1"><a href="https://github.com/waldronlab/TENxIO/commit/50b12aa7b0ec98554c60754ff87c4e438328f24f"><code>waldronlab/TENxIO</code></a></h3>
<p><strong>Message:</strong> use integrated HTML sessionInfo</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 23:17:29 UTC</p>
<hr>
</section>
<section id="waldronlabvisiumio-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabvisiumio-2"><a href="https://github.com/waldronlab/VisiumIO/commit/ca428e7d17c9bb791991413d44c9a64290d9224c"><code>waldronlab/VisiumIO</code></a></h3>
<p><strong>Message:</strong> version bump 1.5.5</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 23:00:43 UTC</p>
<hr>
</section>
<section id="waldronlabvisiumio-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabvisiumio-3"><a href="https://github.com/waldronlab/VisiumIO/commit/a047d6430be0715b65ee2f5e30a623f425b54c56"><code>waldronlab/VisiumIO</code></a></h3>
<p><strong>Message:</strong> remove HTML from .Rbuildignore</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 23:00:32 UTC</p>
<hr>
</section>
<section id="waldronlabvisiumio-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabvisiumio-4"><a href="https://github.com/waldronlab/VisiumIO/commit/fa0e469148bcada0f9ee8bb9afa2c002696b78df"><code>waldronlab/VisiumIO</code></a></h3>
<p><strong>Message:</strong> version bump 1.5.4</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 22:50:54 UTC</p>
<hr>
</section>
<section id="waldronlabxeniumio-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabxeniumio-2"><a href="https://github.com/waldronlab/XeniumIO/commit/302b3c8df75785903fbdd7c230ae241264854c25"><code>waldronlab/XeniumIO</code></a></h3>
<p><strong>Message:</strong> version bump 1.1.2</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 22:23:23 UTC</p>
<hr>
</section>
<section id="waldronlabxeniumio-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabxeniumio-3"><a href="https://github.com/waldronlab/XeniumIO/commit/1eadb5570d00d123536a720555d2ba4f51b494ef"><code>waldronlab/XeniumIO</code></a></h3>
<p><strong>Message:</strong> update README.md</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 22:23:02 UTC</p>
<hr>
</section>
<section id="waldronlabxeniumio-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabxeniumio-4"><a href="https://github.com/waldronlab/XeniumIO/commit/a24595040426f1117bdc5f1bb1bad4ba59651f63"><code>waldronlab/XeniumIO</code></a></h3>
<p><strong>Message:</strong> add expandable sessionInfo in HTML</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 22:22:50 UTC</p>
<hr>
</section>
<section id="waldronlabtenxio-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtenxio-2"><a href="https://github.com/waldronlab/TENxIO/commit/27617012b1b64df6a402697e058269f77feabbff"><code>waldronlab/TENxIO</code></a></h3>
<p><strong>Message:</strong> version bump 1.11.3</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 21:29:03 UTC</p>
<hr>
</section>
<section id="waldronlabtenxio-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtenxio-3"><a href="https://github.com/waldronlab/TENxIO/commit/3350fd40582a59788fb99989e6ec5bcabb135af7"><code>waldronlab/TENxIO</code></a></h3>
<p><strong>Message:</strong> add _pkgdown.yml and .github/workflows/pr_check.yml GHA</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 21:27:10 UTC</p>
<hr>
</section>
<section id="waldronlabtenxio-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtenxio-4"><a href="https://github.com/waldronlab/TENxIO/commit/d2ecffbf934a1fd98d209bb0525392becf6508c4"><code>waldronlab/TENxIO</code></a></h3>
<p><strong>Message:</strong> version bump 1.11.2</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 21:25:17 UTC</p>
<hr>
</section>
<section id="waldronlabsinglecellmultimodal-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabsinglecellmultimodal-2"><a href="https://github.com/waldronlab/SingleCellMultiModal/commit/ccecd624f2e149ad974650ed202ee5addd63fb3c"><code>waldronlab/SingleCellMultiModal</code></a></h3>
<p><strong>Message:</strong> version bump 1.21.2</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 20:21:33 UTC</p>
<hr>
</section>
<section id="waldronlabsinglecellmultimodal-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabsinglecellmultimodal-3"><a href="https://github.com/waldronlab/SingleCellMultiModal/commit/bcb3cb9c589fbf0df60608de3e8e96e0e9edf18a"><code>waldronlab/SingleCellMultiModal</code></a></h3>
<p><strong>Message:</strong> add sticker to README.md</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-06 20:21:25 UTC</p>
<hr>
</section>
<section id="waldronlabsinglecellmultimodal-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabsinglecellmultimodal-4"><a href="https://github.com/waldronlab/SingleCellMultiModal/commit/ee797b5da4b43b07d44e01e3d001b77290fe9240"><code>waldronlab/SingleCellMultiModal</code></a></h3>
<p><strong>Message:</strong> version bump 1.21.1</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 21:43:20 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedtcgadata-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedtcgadata-2"><a href="https://github.com/waldronlab/curatedTCGAData/commit/28d4601ebdd0fe997df5c688fc81c55d51de92d0"><code>waldronlab/curatedTCGAData</code></a></h3>
<p><strong>Message:</strong> version bump 1.31.1</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 21:32:41 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedtcgadata-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedtcgadata-3"><a href="https://github.com/waldronlab/curatedTCGAData/commit/a13b6ab5acf8787846b9279cb1247ce24c955e88"><code>waldronlab/curatedTCGAData</code></a></h3>
<p><strong>Message:</strong> update CI in .Rbuildignore</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 21:31:52 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedtcgadata-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedtcgadata-4"><a href="https://github.com/waldronlab/curatedTCGAData/commit/6c7ebb677a8c2e50374ab0b6f37f0529e3d492b0"><code>waldronlab/curatedTCGAData</code></a></h3>
<p><strong>Message:</strong> add _pkgdown.yml and .github/workflows/pr_check.yml GHA</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 21:23:26 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedovariandata" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedovariandata"><a href="https://github.com/waldronlab/curatedOvarianData/commit/e5f2a11befda9cb0499238d91cbff9972e03a4ed"><code>waldronlab/curatedOvarianData</code></a></h3>
<p><strong>Message:</strong> version bump 1.47.2</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 20:56:03 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedovariandata-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedovariandata-1"><a href="https://github.com/waldronlab/curatedOvarianData/commit/1c4c95fec616edcc147eac64b142af2938d9f4b2"><code>waldronlab/curatedOvarianData</code></a></h3>
<p><strong>Message:</strong> rename CHANGELOG and reformat NEWS.md</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 20:55:56 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedovariandata-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedovariandata-2"><a href="https://github.com/waldronlab/curatedOvarianData/commit/790fc5ad975f2002613b62f0171b5bc9a9bbfa6d"><code>waldronlab/curatedOvarianData</code></a></h3>
<p><strong>Message:</strong> update .Rbuildignore and .gitignore</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 20:55:37 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedovariandata-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedovariandata-3"><a href="https://github.com/waldronlab/curatedOvarianData/commit/12f0dc9efe79ecfd4dadfff3825285531301d896"><code>waldronlab/curatedOvarianData</code></a></h3>
<p><strong>Message:</strong> add R version to fix NOTE</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 20:41:13 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedovariandata-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedovariandata-4"><a href="https://github.com/waldronlab/curatedOvarianData/commit/dbadfc3faee902534a9f65765c25892d43ba9ce2"><code>waldronlab/curatedOvarianData</code></a></h3>
<p><strong>Message:</strong> fix URL in _pkgdown.yml</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 20:41:03 UTC</p>
<hr>
</section>
<section id="bioconductordelayedarray" class="level3">
<h3 class="anchored" data-anchor-id="bioconductordelayedarray"><a href="https://github.com/Bioconductor/DelayedArray/commit/f430b84d082cdb611eaab6c8c3c15cb71996e5a6"><code>Bioconductor/DelayedArray</code></a></h3>
<p><strong>Message:</strong> DelayedArray 0.35.2</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-04-18 05:48:41 UTC</p>
<hr>
</section>
<section id="bioconductordelayedarray-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductordelayedarray-1"><a href="https://github.com/Bioconductor/DelayedArray/commit/af6fb3a86cde174fc2306ba4ff6a26a52ca703aa"><code>Bioconductor/DelayedArray</code></a></h3>
<p><strong>Message:</strong> DelayedArray 0.35.1</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-04-17 17:33:44 UTC</p>
<hr>
</section>
<section id="bioconductorh5mread-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorh5mread-1"><a href="https://github.com/Bioconductor/h5mread/commit/333cad3e32441f725808d33a9f6c9e91f1b62246"><code>Bioconductor/h5mread</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 17:45:26 UTC</p>
<hr>
</section>
<section id="bioconductorh5mread-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorh5mread-2"><a href="https://github.com/Bioconductor/h5mread/commit/cc25660812c2ff8569509ea58fbdfab39a4fcf05"><code>Bioconductor/h5mread</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 17:45:26 UTC</p>
<hr>
</section>
<section id="bioconductorsparsearray-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorsparsearray-2"><a href="https://github.com/Bioconductor/SparseArray/commit/00cb8181f7836e85867ccf982079c7170d60ac7d"><code>Bioconductor/SparseArray</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 17:15:49 UTC</p>
<hr>
</section>
<section id="bioconductorsparsearray-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorsparsearray-3"><a href="https://github.com/Bioconductor/SparseArray/commit/ea36777cc56c1e3b515f936d4adf866dad0a8ced"><code>Bioconductor/SparseArray</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 17:15:49 UTC</p>
<hr>
</section>
<section id="bioconductors4arrays-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductors4arrays-1"><a href="https://github.com/Bioconductor/S4Arrays/commit/b4d3bc757fb6caf05d8d58fac63ff51626c68200"><code>Bioconductor/S4Arrays</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 17:15:41 UTC</p>
<hr>
</section>
<section id="bioconductors4arrays-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductors4arrays-2"><a href="https://github.com/Bioconductor/S4Arrays/commit/9f3b48000da5a73c2c0cff86b0279b6d22dbc146"><code>Bioconductor/S4Arrays</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 17:15:41 UTC</p>
<hr>
</section>
<section id="shbriefgenomicsupersignature-3" class="level3">
<h3 class="anchored" data-anchor-id="shbriefgenomicsupersignature-3"><a href="https://github.com/shbrief/GenomicSuperSignature/commit/56b06dfb47dd0f8c3995118acba6e05331769b99"><code>shbrief/GenomicSuperSignature</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 16:32:36 UTC</p>
<hr>
</section>
<section id="shbriefgenomicsupersignature-4" class="level3">
<h3 class="anchored" data-anchor-id="shbriefgenomicsupersignature-4"><a href="https://github.com/shbrief/GenomicSuperSignature/commit/3a2997410299133760eb32f02abb5dc7fcfd53d9"><code>shbrief/GenomicSuperSignature</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 16:32:36 UTC</p>
<hr>
</section>
<section id="bioconductordelayedarray-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductordelayedarray-2"><a href="https://github.com/Bioconductor/DelayedArray/commit/8ba82b43b1f623f766ea6a0551196b9b0e14c2c3"><code>Bioconductor/DelayedArray</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 15:07:19 UTC</p>
<hr>
</section>
<section id="bioconductordelayedarray-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductordelayedarray-3"><a href="https://github.com/Bioconductor/DelayedArray/commit/45b9940d40d76224026fb4f0b66bf7c6ce273fbb"><code>Bioconductor/DelayedArray</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 15:07:19 UTC</p>
<hr>
</section>
<section id="bioconductorhdf5array" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorhdf5array"><a href="https://github.com/Bioconductor/HDF5Array/commit/2d9647e8067d666ddaabd7d1ad398a2dd1f6caa8"><code>Bioconductor/HDF5Array</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 14:58:57 UTC</p>
<hr>
</section>
<section id="bioconductorhdf5array-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorhdf5array-1"><a href="https://github.com/Bioconductor/HDF5Array/commit/1e32d9f9fc78bddffd1600aee8e46d76fa296b71"><code>Bioconductor/HDF5Array</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-04-15 14:58:57 UTC</p>
<hr>
</section>
<section id="bioconductordelayedarray-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductordelayedarray-4"><a href="https://github.com/Bioconductor/DelayedArray/commit/cb6a7b742b52f39fcc174489493b44354125f3ba"><code>Bioconductor/DelayedArray</code></a></h3>
<p><strong>Message:</strong> update NEWS</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-04-15 06:49:59 UTC</p>
<hr>
</section>
<section id="bioconductors4arrays-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductors4arrays-3"><a href="https://github.com/Bioconductor/S4Arrays/commit/9b6e35aabe01c2d3eb0ff1fc56767bab02336936"><code>Bioconductor/S4Arrays</code></a></h3>
<p><strong>Message:</strong> update NEWS</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-04-15 06:41:28 UTC</p>
<hr>
</section>
<section id="bioconductorsparsearray-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorsparsearray-4"><a href="https://github.com/Bioconductor/SparseArray/commit/4ea70457f06221a03075ce4e626674673f7511db"><code>Bioconductor/SparseArray</code></a></h3>
<p><strong>Message:</strong> update NEWS</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-04-15 05:13:09 UTC</p>
<hr>
</section>
<section id="bioconductorhdf5array-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorhdf5array-2"><a href="https://github.com/Bioconductor/HDF5Array/commit/e087336a0f4e7e408f04f1ed7d5fa95a936b706c"><code>Bioconductor/HDF5Array</code></a></h3>
<p><strong>Message:</strong> update NEWS</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-04-15 01:35:24 UTC</p>
<hr>
</section>
<section id="bioconductorhdf5array-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorhdf5array-3"><a href="https://github.com/Bioconductor/HDF5Array/commit/e9399298bc98f9907bece2446393140e18bfb250"><code>Bioconductor/HDF5Array</code></a></h3>
<p><strong>Message:</strong> HDF5Array 1.35.17</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-04-15 01:12:05 UTC</p>
<hr>
</section>
<section id="hpageshovernethelp-2" class="level3">
<h3 class="anchored" data-anchor-id="hpageshovernethelp-2"><a href="https://github.com/hpages/hovernethelp/commit/5dc64f1c389ac93efa57ad3809d6c6977b94f9f1"><code>hpages/hovernethelp</code></a></h3>
<p><strong>Message:</strong> update minicluster/JOURNAL_OF_RUNS.txt</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-04-14 17:38:58 UTC</p>
<hr>
</section>
<section id="qiicrdcmqi-3" class="level3">
<h3 class="anchored" data-anchor-id="qiicrdcmqi-3"><a href="https://github.com/QIICR/dcmqi/commit/4e5b700bc97ef9da2cf4937f008a165338899a8f"><code>QIICR/dcmqi</code></a></h3>
<p><strong>Message:</strong> cmake: update version to 1.4.0</p>
<p><strong>Author:</strong> Andrey Fedorov</p>
<p><strong>Date:</strong> 2025-04-07 19:55:16 UTC</p>
<hr>
</section>
<section id="qiicrdcmqi-4" class="level3">
<h3 class="anchored" data-anchor-id="qiicrdcmqi-4"><a href="https://github.com/QIICR/dcmqi/commit/c69a74248e43c8abe86d927af7f4724167fb585d"><code>QIICR/dcmqi</code></a></h3>
<p><strong>Message:</strong> Merge pull request #500 from QIICR/latest-dcmtk</p>
<p><strong>Author:</strong> Andrey Fedorov</p>
<p><strong>Date:</strong> 2025-04-07 16:55:13 UTC</p>
<hr>
</section>
<section id="hpageshovernethelp-3" class="level3">
<h3 class="anchored" data-anchor-id="hpageshovernethelp-3"><a href="https://github.com/hpages/hovernethelp/commit/08f6eaeb6a13b472e8cecbf7fc83d088b3401f44"><code>hpages/hovernethelp</code></a></h3>
<p><strong>Message:</strong> infer_batch2.sh now pushes results to inferdata4 volume instead of inferdata3</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-04-01 15:41:17 UTC</p>
<hr>
</section>
<section id="hpageshovernethelp-4" class="level3">
<h3 class="anchored" data-anchor-id="hpageshovernethelp-4"><a href="https://github.com/hpages/hovernethelp/commit/d9ad821c06f58f7a443ca9c4cd25a81f84dae478"><code>hpages/hovernethelp</code></a></h3>
<p><strong>Message:</strong> update minicluster/JOURNAL_OF_RUNS.txt</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-03-25 18:49:02 UTC</p>
<hr>
</section>
<section id="seandavibiocbuilddb" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocbuilddb"><a href="https://github.com/seandavi/BiocBuildDB/commit/bd5b01e25e7edbae810bc02d280a9b7dceee404c"><code>seandavi/BiocBuildDB</code></a></h3>
<p><strong>Message:</strong> Clarify that the package is used to help build a database….</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-03-20 02:00:22 UTC</p>
<hr>
</section>
<section id="bioconductorhdf5array-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorhdf5array-4"><a href="https://github.com/Bioconductor/HDF5Array/commit/34b62190897b98d7337f9be06292fe5640454972"><code>Bioconductor/HDF5Array</code></a></h3>
<p><strong>Message:</strong> minor edit</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-03-19 22:22:56 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm-2" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm-2"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/e48c5a49175c30130d0bea43efc6ff6fbac3c85a"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> Update body_site_details</p>
<p><strong>Author:</strong> krlong68</p>
<p><strong>Date:</strong> 2025-03-18 20:02:13 UTC</p>
<hr>
</section>
<section id="seandaviawesome-cancer-variant-resources" class="level3">
<h3 class="anchored" data-anchor-id="seandaviawesome-cancer-variant-resources"><a href="https://github.com/seandavi/awesome-cancer-variant-resources/commit/493b8123d5adaf998147e8720dd463b287ce0c09"><code>seandavi/awesome-cancer-variant-resources</code></a></h3>
<p><strong>Message:</strong> Merge pull request #7 from lgeistlinger/master</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-03-06 23:04:53 UTC</p>
<hr>
</section>
<section id="seandaviawesome-cancer-variant-resources-1" class="level3">
<h3 class="anchored" data-anchor-id="seandaviawesome-cancer-variant-resources-1"><a href="https://github.com/seandavi/awesome-cancer-variant-resources/commit/cc98cf75f888701836a4a64c18c1dea2ad87a5f7"><code>seandavi/awesome-cancer-variant-resources</code></a></h3>
<p><strong>Message:</strong> add AlphaMissense</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2025-03-06 22:14:12 UTC</p>
<hr>
</section>
<section id="seandavirefinebioc" class="level3">
<h3 class="anchored" data-anchor-id="seandavirefinebioc"><a href="https://github.com/seandavi/refinebioc/commit/33131f2c6d5be1870d3d82d2386418c3f2a25608"><code>seandavi/refinebioc</code></a></h3>
<p><strong>Message:</strong> Merge pull request #17 from ttriche/patch-1</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-03-04 23:28:32 UTC</p>
<hr>
</section>
<section id="bioconductorbiochubsingestr" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiochubsingestr"><a href="https://github.com/Bioconductor/BiocHubsIngestR/commit/09895350349bc559ccbd27dbc4b9d1ee1f5d63d6"><code>Bioconductor/BiocHubsIngestR</code></a></h3>
<p><strong>Message:</strong> Bump version</p>
<p><strong>Author:</strong> Alexandru Mahmoud</p>
<p><strong>Date:</strong> 2025-02-27 17:00:06 UTC</p>
<hr>
</section>
<section id="seandavirefinebioc-1" class="level3">
<h3 class="anchored" data-anchor-id="seandavirefinebioc-1"><a href="https://github.com/seandavi/refinebioc/commit/4e17ad816620568952e04bd5afd7d44e37e5d01c"><code>seandavi/refinebioc</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Tim Triche, Jr.</p>
<p><strong>Date:</strong> 2025-02-27 16:02:51 UTC</p>
<hr>
</section>
<section id="bioconductorbiochubsingestr-1" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiochubsingestr-1"><a href="https://github.com/Bioconductor/BiocHubsIngestR/commit/a6bc6436e8f9ce2d9739a4a479c996f3efe6d8ab"><code>Bioconductor/BiocHubsIngestR</code></a></h3>
<p><strong>Message:</strong> Update part size</p>
<p><strong>Author:</strong> Alexandru Mahmoud</p>
<p><strong>Date:</strong> 2025-02-27 15:19:15 UTC</p>
<hr>
</section>
<section id="bioconductorbiochubsingestr-2" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiochubsingestr-2"><a href="https://github.com/Bioconductor/BiocHubsIngestR/commit/6ddc4e7e9dace66f14f9d3e00c927ef886cc5265"><code>Bioconductor/BiocHubsIngestR</code></a></h3>
<p><strong>Message:</strong> Test multi-part upload</p>
<p><strong>Author:</strong> Alexandru Mahmoud</p>
<p><strong>Date:</strong> 2025-02-26 19:27:57 UTC</p>
<hr>
</section>
<section id="seandavirefinebioc-2" class="level3">
<h3 class="anchored" data-anchor-id="seandavirefinebioc-2"><a href="https://github.com/seandavi/refinebioc/commit/d1f79a6f49a016337707a833824ac2a2943e5b52"><code>seandavi/refinebioc</code></a></h3>
<p><strong>Message:</strong> try fixing email on gh actions</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-02-21 04:03:51 UTC</p>
<hr>
</section>
<section id="seandavirefinebioc-3" class="level3">
<h3 class="anchored" data-anchor-id="seandavirefinebioc-3"><a href="https://github.com/seandavi/refinebioc/commit/a7b15f59b37150f9b6997f3fe9fc115375758a3d"><code>seandavi/refinebioc</code></a></h3>
<p><strong>Message:</strong> replace knitr with quarto</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-02-21 03:41:49 UTC</p>
<hr>
</section>
<section id="seandavirefinebioc-4" class="level3">
<h3 class="anchored" data-anchor-id="seandavirefinebioc-4"><a href="https://github.com/seandavi/refinebioc/commit/e34c4b03c2880e925a004e623ca097d760d25fcd"><code>seandavi/refinebioc</code></a></h3>
<p><strong>Message:</strong> fix vignette build</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-02-21 03:37:35 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm-3" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm-3"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/96746a3d403dc846eb9794db0c3b9069b5d13931"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> Add dictionaries for “ancestry_details”, “body_site_details”, and “disease_details”; add Svetlana to curator list</p>
<p><strong>Author:</strong> krlong68</p>
<p><strong>Date:</strong> 2025-02-18 19:09:29 UTC</p>
<hr>
</section>
<section id="seandavistate-cancer-profile-scraper" class="level3">
<h3 class="anchored" data-anchor-id="seandavistate-cancer-profile-scraper"><a href="https://github.com/seandavi/state-cancer-profile-scraper/commit/a292cca9e46abed098f9b25793d8c891cd3e3c36"><code>seandavi/state-cancer-profile-scraper</code></a></h3>
<p><strong>Message:</strong> Clarify readme intro</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-02-11 00:18:39 UTC</p>
<hr>
</section>
<section id="seandavistate-cancer-profile-scraper-1" class="level3">
<h3 class="anchored" data-anchor-id="seandavistate-cancer-profile-scraper-1"><a href="https://github.com/seandavi/state-cancer-profile-scraper/commit/49ccc97937a5c048073e3d252a19311b7b2459fd"><code>seandavi/state-cancer-profile-scraper</code></a></h3>
<p><strong>Message:</strong> Add update frequency</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-02-10 23:02:05 UTC</p>
<hr>
</section>
<section id="seandavistate-cancer-profile-scraper-2" class="level3">
<h3 class="anchored" data-anchor-id="seandavistate-cancer-profile-scraper-2"><a href="https://github.com/seandavi/state-cancer-profile-scraper/commit/1d5c15d964ed9337a29b0dbd0eb7db4bc2676ce1"><code>seandavi/state-cancer-profile-scraper</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-02-10 22:58:20 UTC</p>
<hr>
</section>
<section id="seandavistate-cancer-profile-scraper-3" class="level3">
<h3 class="anchored" data-anchor-id="seandavistate-cancer-profile-scraper-3"><a href="https://github.com/seandavi/state-cancer-profile-scraper/commit/702027ef78cf84dbd413f05887b2c8303a3cebf5"><code>seandavi/state-cancer-profile-scraper</code></a></h3>
<p><strong>Message:</strong> Add clear download link.</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-02-10 22:46:07 UTC</p>
<hr>
</section>
<section id="seandavistate-cancer-profile-scraper-4" class="level3">
<h3 class="anchored" data-anchor-id="seandavistate-cancer-profile-scraper-4"><a href="https://github.com/seandavi/state-cancer-profile-scraper/commit/85998019dba1b509048a53e9a5020ce361cf79a1"><code>seandavi/state-cancer-profile-scraper</code></a></h3>
<p><strong>Message:</strong> update ignores</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-02-10 22:41:38 UTC</p>
<hr>
</section>
<section id="bioconductors4arrays-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductors4arrays-4"><a href="https://github.com/Bioconductor/S4Arrays/commit/41a64ffe40d2e1d2d7092631794de918b5fdab2c"><code>Bioconductor/S4Arrays</code></a></h3>
<p><strong>Message:</strong> S4Arrays 1.7.3</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-02-08 02:02:59 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm-4" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm-4"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/c09d3eee3791a9319a001da8b87b5e7c6a430c7e"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> Add non-NCIT terms to “treatment” dictionary; create .owl files for “probing_pocket_depth”, “smoker”, and “feces_phenotype” terms</p>
<p><strong>Author:</strong> krlong68</p>
<p><strong>Date:</strong> 2025-02-07 13:12:20 UTC</p>
<hr>
</section>
<section id="bioconductorbiochubsingestr-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiochubsingestr-3"><a href="https://github.com/Bioconductor/BiocHubsIngestR/commit/a3b6ccaebd4b1429d52514e004d9af7bbbd55cee"><code>Bioconductor/BiocHubsIngestR</code></a></h3>
<p><strong>Message:</strong> Tweaks</p>
<p><strong>Author:</strong> Alexandru Mahmoud</p>
<p><strong>Date:</strong> 2025-01-23 19:29:26 UTC</p>
<hr>
</section>
<section id="bioconductorh5mread-3" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorh5mread-3"><a href="https://github.com/Bioconductor/h5mread/commit/8aced109b792a4a8c807481d6efc5e0d766004d4"><code>Bioconductor/h5mread</code></a></h3>
<p><strong>Message:</strong> h5mread 0.99.4</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-01-21 21:44:07 UTC</p>
<hr>
</section>
<section id="bioconductorh5mread-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorh5mread-4"><a href="https://github.com/Bioconductor/h5mread/commit/8d58c0947e539b76ab7336e4779e66157fb9db2b"><code>Bioconductor/h5mread</code></a></h3>
<p><strong>Message:</strong> minor edit to vignette</p>
<p><strong>Author:</strong> Hervé Pagès</p>
<p><strong>Date:</strong> 2025-01-14 22:39:31 UTC</p>
<hr>
</section>
<section id="bioconductorbiochubsingestr-4" class="level3">
<h3 class="anchored" data-anchor-id="bioconductorbiochubsingestr-4"><a href="https://github.com/Bioconductor/BiocHubsIngestR/commit/3c8b3d02727e3dba09ce4f3830c6e3f039de84e2"><code>Bioconductor/BiocHubsIngestR</code></a></h3>
<p><strong>Message:</strong> Remove License added by tempalte</p>
<p><strong>Author:</strong> Alexandru Mahmoud</p>
<p><strong>Date:</strong> 2025-01-10 20:59:10 UTC</p>
<hr>
</section>
<section id="waldronlabpublicdataresources" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabpublicdataresources"><a href="https://github.com/waldronlab/PublicDataResources/commit/7a96b5ddb94a4cde40816e320fb7408e75f06855"><code>waldronlab/PublicDataResources</code></a></h3>
<p><strong>Message:</strong> set actions/cache to v4</p>
<p><strong>Author:</strong> Marcel Ramos Pérez</p>
<p><strong>Date:</strong> 2025-01-06 21:26:21 UTC</p>
<hr>
</section>
<section id="waldronlabenrichomics" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabenrichomics"><a href="https://github.com/waldronlab/enrichOmics/commit/cc4ae720d4a1500f83b7b70f8114ec00eb28e7b1"><code>waldronlab/enrichOmics</code></a></h3>
<p><strong>Message:</strong> set actions/cache to v4</p>
<p><strong>Author:</strong> Marcel Ramos Pérez</p>
<p><strong>Date:</strong> 2025-01-06 21:24:40 UTC</p>
<hr>
</section>
<section id="vjcitnxenlite" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnxenlite"><a href="https://github.com/vjcitn/xenLite/commit/97b2b8e20b714dc60078fcfb0fbd1d9240ff257e"><code>vjcitn/xenLite</code></a></h3>
<p><strong>Message:</strong> Merge pull request #4 from vjcitn/devel</p>
<p><strong>Author:</strong> Vince Carey</p>
<p><strong>Date:</strong> 2024-11-27 11:38:02 UTC</p>
<hr>
</section>
<section id="vjcitnxenlite-1" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnxenlite-1"><a href="https://github.com/vjcitn/xenLite/commit/4bfc2f89020e462eea6823d08d561f11db2f70bf"><code>vjcitn/xenLite</code></a></h3>
<p><strong>Message:</strong> Merge pull request #3 from vjcitn/vincedev</p>
<p><strong>Author:</strong> Vince Carey</p>
<p><strong>Date:</strong> 2024-11-27 11:36:00 UTC</p>
<hr>
</section>
<section id="waldronlabhgnchelper" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgnchelper"><a href="https://github.com/waldronlab/HGNChelper/commit/42a787e43cb5cba0299b8193e6b2a28861ad5791"><code>waldronlab/HGNChelper</code></a></h3>
<p><strong>Message:</strong> update NEWS</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2024-11-16 16:08:23 UTC</p>
<hr>
</section>
<section id="waldronlabhgnchelper-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgnchelper-1"><a href="https://github.com/waldronlab/HGNChelper/commit/0874f323f606bcd94772361c2cc212abe2827407"><code>waldronlab/HGNChelper</code></a></h3>
<p><strong>Message:</strong> Bump version, update maps, suppress warning in splitstachshape::cSplit, update HGNC URL in documentation</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2024-11-16 15:53:39 UTC</p>
<hr>
</section>
<section id="waldronlabhgnchelper-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgnchelper-2"><a href="https://github.com/waldronlab/HGNChelper/commit/b055e04938ec079ac158ed0661391bcde63f4530"><code>waldronlab/HGNChelper</code></a></h3>
<p><strong>Message:</strong> update url, refers to #1</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2024-11-11 15:02:53 UTC</p>
<hr>
</section>
<section id="vjcitnxenlite-2" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnxenlite-2"><a href="https://github.com/vjcitn/xenLite/commit/2a0668358a2df0c55e2fbd5e73910a8029dafb25"><code>vjcitn/xenLite</code></a></h3>
<p><strong>Message:</strong> Merge pull request #2 from vjcitn/RELEASE_3_20</p>
<p><strong>Author:</strong> Vince Carey</p>
<p><strong>Date:</strong> 2024-10-31 05:07:15 UTC</p>
<hr>
</section>
<section id="vjcitnyescds" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnyescds"><a href="https://github.com/vjcitn/YESCDS/commit/e089d143478245201c359ebfc6d09d101de4b910"><code>vjcitn/YESCDS</code></a></h3>
<p><strong>Message:</strong> update the pkgdown site</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2024-10-30 12:26:08 UTC</p>
<hr>
</section>
<section id="vjcitnxenlite-3" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnxenlite-3"><a href="https://github.com/vjcitn/xenLite/commit/921c0573b73fd219fc2360a6435d072755028916"><code>vjcitn/xenLite</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_20 branch</p>
<p><strong>Author:</strong> J Wokaty</p>
<p><strong>Date:</strong> 2024-10-29 15:33:39 UTC</p>
<hr>
</section>
<section id="vjcitnxenlite-4" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnxenlite-4"><a href="https://github.com/vjcitn/xenLite/commit/f59dc8b1d9a20186f46d17596ec245467e54366a"><code>vjcitn/xenLite</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_20 branch</p>
<p><strong>Author:</strong> J Wokaty</p>
<p><strong>Date:</strong> 2024-10-29 15:33:39 UTC</p>
<hr>
</section>
<section id="vjcitnyescds-1" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnyescds-1"><a href="https://github.com/vjcitn/YESCDS/commit/4ccd488629dd002b1e7bd3df97fd941092417483"><code>vjcitn/YESCDS</code></a></h3>
<p><strong>Message:</strong> passing check</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2024-08-02 14:05:14 UTC</p>
<hr>
</section>
<section id="vjcitnyescds-2" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnyescds-2"><a href="https://github.com/vjcitn/YESCDS/commit/80d191b124ac549c07d70589aa1a7bd275378308"><code>vjcitn/YESCDS</code></a></h3>
<p><strong>Message:</strong> moving along, dropping mut_stuff for now</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2024-08-02 11:34:25 UTC</p>
<hr>
</section>
<section id="vjcitnyescds-3" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnyescds-3"><a href="https://github.com/vjcitn/YESCDS/commit/926f5cf14ad8dea6495ab82381f90dedc86b8b75"><code>vjcitn/YESCDS</code></a></h3>
<p><strong>Message:</strong> update compressed data</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2024-08-02 11:24:15 UTC</p>
<hr>
</section>
<section id="vjcitnyescds-4" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnyescds-4"><a href="https://github.com/vjcitn/YESCDS/commit/ccd5bc2d1ef46e7c732b13272b0a2085a15881ac"><code>vjcitn/YESCDS</code></a></h3>
<p><strong>Message:</strong> adding hcc code causing problem with devtools::document?</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2024-08-02 09:50:51 UTC</p>
<hr>
</section>
<section id="ccb-hmsscconform" class="level3">
<h3 class="anchored" data-anchor-id="ccb-hmsscconform"><a href="https://github.com/ccb-hms/scConform/commit/e1feb4fb1473e0d1ec760a16254fe01545f6bd01"><code>ccb-hms/scConform</code></a></h3>
<p><strong>Message:</strong> change param names and add ctb</p>
<p><strong>Author:</strong> DanielaCorbetta</p>
<p><strong>Date:</strong> 2024-07-25 15:02:51 UTC</p>
<hr>
</section>
<section id="ccb-hmsscconform-1" class="level3">
<h3 class="anchored" data-anchor-id="ccb-hmsscconform-1"><a href="https://github.com/ccb-hms/scConform/commit/5253658e421b11b8596e9e45ae941c4ad4871a84"><code>ccb-hms/scConform</code></a></h3>
<p><strong>Message:</strong> add pkgdown css</p>
<p><strong>Author:</strong> DanielaCorbetta</p>
<p><strong>Date:</strong> 2024-07-02 20:17:11 UTC</p>
<hr>
</section>
<section id="ccb-hmsscconform-2" class="level3">
<h3 class="anchored" data-anchor-id="ccb-hmsscconform-2"><a href="https://github.com/ccb-hms/scConform/commit/c3862e1741abfc1fdd57c0b5d4af8db9e4c29f4b"><code>ccb-hms/scConform</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘main’ of https://github.com/ccb-hms/scConform</p>
<p><strong>Author:</strong> DanielaCorbetta</p>
<p><strong>Date:</strong> 2024-06-24 19:10:27 UTC</p>
<hr>
</section>
<section id="ccb-hmsscconform-3" class="level3">
<h3 class="anchored" data-anchor-id="ccb-hmsscconform-3"><a href="https://github.com/ccb-hms/scConform/commit/930572cecbf8de2f279a3597c7de1df41ec05adb"><code>ccb-hms/scConform</code></a></h3>
<p><strong>Message:</strong> rm import foreach</p>
<p><strong>Author:</strong> DanielaCorbetta</p>
<p><strong>Date:</strong> 2024-06-24 19:10:25 UTC</p>
<hr>
</section>
<section id="ccb-hmsscconform-4" class="level3">
<h3 class="anchored" data-anchor-id="ccb-hmsscconform-4"><a href="https://github.com/ccb-hms/scConform/commit/2369e8c3260b5f7c608ae1f221de0cd797612f0b"><code>ccb-hms/scConform</code></a></h3>
<p><strong>Message:</strong> Update check-bioc.yml</p>
<p><strong>Author:</strong> Daniela Corbetta</p>
<p><strong>Date:</strong> 2024-06-24 16:49:55 UTC</p>
<hr>
</section>
<section id="waldronlabhgnchelper-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgnchelper-3"><a href="https://github.com/waldronlab/HGNChelper/commit/a9619a2abfa7c6f1e0331b0c9c0cd7c16886eeeb"><code>waldronlab/HGNChelper</code></a></h3>
<p><strong>Message:</strong> version bump</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2024-05-18 04:16:43 UTC</p>
<hr>
</section>
<section id="waldronlabhgnchelper-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhgnchelper-4"><a href="https://github.com/waldronlab/HGNChelper/commit/d53cc84f1113793b551c2f56d1e06411d7ff893a"><code>waldronlab/HGNChelper</code></a></h3>
<p><strong>Message:</strong> Documentation syntax</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2024-05-18 04:16:35 UTC</p>
<hr>
</section>
<section id="seandavibiocbuilddb-1" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocbuilddb-1"><a href="https://github.com/seandavi/BiocBuildDB/commit/25bdab41d175cfc5d1e824be885ec38854f74d7c"><code>seandavi/BiocBuildDB</code></a></h3>
<p><strong>Message:</strong> Update process_new_build_reports.yaml</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2024-02-29 18:07:08 UTC</p>
<hr>
</section>
<section id="seandavibiocbuilddb-2" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocbuilddb-2"><a href="https://github.com/seandavi/BiocBuildDB/commit/ca70379b942ec904abc3f1c091181e4fc61731fa"><code>seandavi/BiocBuildDB</code></a></h3>
<p><strong>Message:</strong> Update process_new_build_reports.yaml</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2024-02-29 18:04:58 UTC</p>
<hr>
</section>
<section id="seandavibiocbuilddb-3" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocbuilddb-3"><a href="https://github.com/seandavi/BiocBuildDB/commit/1abe25a96112e626c878a73b11b333a948c2f20c"><code>seandavi/BiocBuildDB</code></a></h3>
<p><strong>Message:</strong> Update process_new_build_reports.yaml</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2024-02-29 17:59:24 UTC</p>
<hr>
</section>
<section id="seandavibiocbuilddb-4" class="level3">
<h3 class="anchored" data-anchor-id="seandavibiocbuilddb-4"><a href="https://github.com/seandavi/BiocBuildDB/commit/0bcb929b7408f31d2eda92e86bd8e6248991ee19"><code>seandavi/BiocBuildDB</code></a></h3>
<p><strong>Message:</strong> Update process_new_build_reports.yaml</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2024-02-28 23:23:34 UTC</p>
<hr>
</section>
<section id="seandaviawesome-cancer-variant-resources-2" class="level3">
<h3 class="anchored" data-anchor-id="seandaviawesome-cancer-variant-resources-2"><a href="https://github.com/seandavi/awesome-cancer-variant-resources/commit/8f7fcfc59f2bcfff4da51f7f6fa400fcace6c590"><code>seandavi/awesome-cancer-variant-resources</code></a></h3>
<p><strong>Message:</strong> Merge pull request #6 from CompOnco/master</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2023-10-24 15:18:04 UTC</p>
<hr>
</section>
<section id="seandaviawesome-cancer-variant-resources-3" class="level3">
<h3 class="anchored" data-anchor-id="seandaviawesome-cancer-variant-resources-3"><a href="https://github.com/seandavi/awesome-cancer-variant-resources/commit/20fd0ad4202d755610a3fbb3f2131e3e76786eed"><code>seandavi/awesome-cancer-variant-resources</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Aedin Culhane</p>
<p><strong>Date:</strong> 2023-10-24 15:04:59 UTC</p>
<hr>
</section>
<section id="cornerstonejsdicomparser" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejsdicomparser"><a href="https://github.com/cornerstonejs/dicomParser/commit/7d2084349bf2bdaffe74021e27b286a6c295ca66"><code>cornerstonejs/dicomParser</code></a></h3>
<p><strong>Message:</strong> Merge pull request #261 from cornerstonejs/dependabot/npm_and_yarn/babel/traverse-7.23.2</p>
<p><strong>Author:</strong> yagni</p>
<p><strong>Date:</strong> 2023-10-17 21:50:44 UTC</p>
<hr>
</section>
<section id="cornerstonejsdicomparser-1" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejsdicomparser-1"><a href="https://github.com/cornerstonejs/dicomParser/commit/cd622a6d9b8430687eaeb7b5696df619d49ea019"><code>cornerstonejs/dicomParser</code></a></h3>
<p><strong>Message:</strong> chore(deps-dev): bump <span class="citation" data-cites="babel/traverse">@babel/traverse</span> from 7.17.0 to 7.23.2</p>
<p><strong>Author:</strong> dependabot[bot]</p>
<p><strong>Date:</strong> 2023-10-17 16:09:07 UTC</p>
<hr>
</section>
<section id="cornerstonejsdicomparser-2" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejsdicomparser-2"><a href="https://github.com/cornerstonejs/dicomParser/commit/7fbbe7492ad167a676f1699dcdeb75f9df875a87"><code>cornerstonejs/dicomParser</code></a></h3>
<p><strong>Message:</strong> Merge pull request #260 from cornerstonejs/dependabot/npm_and_yarn/word-wrap-1.2.5</p>
<p><strong>Author:</strong> yagni</p>
<p><strong>Date:</strong> 2023-08-21 14:05:27 UTC</p>
<hr>
</section>
<section id="cornerstonejsdicomparser-3" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejsdicomparser-3"><a href="https://github.com/cornerstonejs/dicomParser/commit/b686a23a5226bf3730d87dfd4d8b5e7132075d64"><code>cornerstonejs/dicomParser</code></a></h3>
<p><strong>Message:</strong> chore(deps-dev): bump word-wrap from 1.2.3 to 1.2.5</p>
<p><strong>Author:</strong> dependabot[bot]</p>
<p><strong>Date:</strong> 2023-08-21 14:03:17 UTC</p>
<hr>
</section>
<section id="cornerstonejsdicomparser-4" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejsdicomparser-4"><a href="https://github.com/cornerstonejs/dicomParser/commit/c8abdd410cdc286000a661faa5cb9ef3fa5f91b8"><code>cornerstonejs/dicomParser</code></a></h3>
<p><strong>Message:</strong> Merge pull request #259 from yagni/update-browser-tools</p>
<p><strong>Author:</strong> yagni</p>
<p><strong>Date:</strong> 2023-08-21 14:01:01 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonetools" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonetools"><a href="https://github.com/cornerstonejs/cornerstoneTools/commit/dbf874f744a91fcf3242659052d6d64b84bb61a7"><code>cornerstonejs/cornerstoneTools</code></a></h3>
<p><strong>Message:</strong> Merge pull request #1550 from ViniciusResende/fix/seg-rendering-rescaling</p>
<p><strong>Author:</strong> Igor Octaviano</p>
<p><strong>Date:</strong> 2023-07-21 11:26:41 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonetools-1" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonetools-1"><a href="https://github.com/cornerstonejs/cornerstoneTools/commit/4363285db54820972adf143faf54b138c53e0677"><code>cornerstonejs/cornerstoneTools</code></a></h3>
<p><strong>Message:</strong> improvement(SegRendering): added unit test file to test renderFill function</p>
<p><strong>Author:</strong> Vinícius Alves</p>
<p><strong>Date:</strong> 2023-07-21 11:06:20 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonetools-2" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonetools-2"><a href="https://github.com/cornerstonejs/cornerstoneTools/commit/27bfacc2ee65612edd1a511cc01ebe7f64528e32"><code>cornerstonejs/cornerstoneTools</code></a></h3>
<p><strong>Message:</strong> fix(SegRendering): preserve canvas transform matrix after seg render</p>
<p><strong>Author:</strong> Vinícius Alves</p>
<p><strong>Date:</strong> 2023-07-21 11:04:48 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonewadoimageloader" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonewadoimageloader"><a href="https://github.com/cornerstonejs/cornerstoneWADOImageLoader/commit/2d947aff9d8430bd9c1a1ee4c3a1d105e65708f3"><code>cornerstonejs/cornerstoneWADOImageLoader</code></a></h3>
<p><strong>Message:</strong> fix: copy buffer to prevent oom in workers (#454)</p>
<p><strong>Author:</strong> Ouwen Huang</p>
<p><strong>Date:</strong> 2023-06-14 03:09:47 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonetools-3" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonetools-3"><a href="https://github.com/cornerstonejs/cornerstoneTools/commit/73cd5224782cc757275b495178c7889ba95eb152"><code>cornerstonejs/cornerstoneTools</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Alireza</p>
<p><strong>Date:</strong> 2023-06-12 17:58:00 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone"><a href="https://github.com/cornerstonejs/cornerstone/commit/da4727841d56872b78ae60f4b7a1e58468b7fad6"><code>cornerstonejs/cornerstone</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Alireza</p>
<p><strong>Date:</strong> 2023-06-12 17:57:43 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonetools-4" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonetools-4"><a href="https://github.com/cornerstonejs/cornerstoneTools/commit/b38c944fe98f213f6128d4825041f0ba2d988a97"><code>cornerstonejs/cornerstoneTools</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Alireza</p>
<p><strong>Date:</strong> 2023-06-12 17:57:06 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone-1" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone-1"><a href="https://github.com/cornerstonejs/cornerstone/commit/a43669426326d764c9def7736396a687f2e32805"><code>cornerstonejs/cornerstone</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Alireza</p>
<p><strong>Date:</strong> 2023-06-12 17:56:14 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment.tcga" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment.tcga"><a href="https://github.com/waldronlab/MultiAssayExperiment.TCGA/commit/bbde8f30879135bf0625fdd34b86b0daa1c437d0"><code>waldronlab/MultiAssayExperiment.TCGA</code></a></h3>
<p><strong>Message:</strong> standardize barcodes for curation files</p>
<p><strong>Author:</strong> LiNK-NY</p>
<p><strong>Date:</strong> 2023-05-16 22:14:53 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment.tcga-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment.tcga-1"><a href="https://github.com/waldronlab/MultiAssayExperiment.TCGA/commit/8fc1590bc0fce1c335af0e007ced9e52de78492d"><code>waldronlab/MultiAssayExperiment.TCGA</code></a></h3>
<p><strong>Message:</strong> use file instead of path</p>
<p><strong>Author:</strong> LiNK-NY</p>
<p><strong>Date:</strong> 2023-05-16 22:14:30 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment.tcga-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment.tcga-2"><a href="https://github.com/waldronlab/MultiAssayExperiment.TCGA/commit/96c643c0eb1ed39cb0c9ef79ceb428400663b8a6"><code>waldronlab/MultiAssayExperiment.TCGA</code></a></h3>
<p><strong>Message:</strong> add version 2.1.1 section</p>
<p><strong>Author:</strong> LiNK-NY</p>
<p><strong>Date:</strong> 2023-05-02 19:14:16 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment.tcga-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment.tcga-3"><a href="https://github.com/waldronlab/MultiAssayExperiment.TCGA/commit/b1ef9985afeb79b3d97d66514eb4078eced78078"><code>waldronlab/MultiAssayExperiment.TCGA</code></a></h3>
<p><strong>Message:</strong> allow flexible runs for only colData</p>
<p><strong>Author:</strong> LiNK-NY</p>
<p><strong>Date:</strong> 2023-05-02 19:10:57 UTC</p>
<hr>
</section>
<section id="waldronlabmultiassayexperiment.tcga-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmultiassayexperiment.tcga-4"><a href="https://github.com/waldronlab/MultiAssayExperiment.TCGA/commit/cc5b93ca68f541b345793f1ff67de0852be3f9e6"><code>waldronlab/MultiAssayExperiment.TCGA</code></a></h3>
<p><strong>Message:</strong> minor updates</p>
<p><strong>Author:</strong> LiNK-NY</p>
<p><strong>Date:</strong> 2023-05-02 19:09:53 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonewadoimageloader-1" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonewadoimageloader-1"><a href="https://github.com/cornerstonejs/cornerstoneWADOImageLoader/commit/093db6714c2593927bb764a0640e59bca4a6a41f"><code>cornerstonejs/cornerstoneWADOImageLoader</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Alireza</p>
<p><strong>Date:</strong> 2023-04-26 18:16:10 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonewadoimageloader-2" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonewadoimageloader-2"><a href="https://github.com/cornerstonejs/cornerstoneWADOImageLoader/commit/fab3c4fcdc5e5ed688e70a27acb60004fc96fc36"><code>cornerstonejs/cornerstoneWADOImageLoader</code></a></h3>
<p><strong>Message:</strong> fix(suv display): fix scaling of non-SUV PT images (#526)</p>
<p><strong>Author:</strong> Patrick Wespi</p>
<p><strong>Date:</strong> 2023-04-14 02:19:47 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonewadoimageloader-3" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonewadoimageloader-3"><a href="https://github.com/cornerstonejs/cornerstoneWADOImageLoader/commit/b8092fc20f6cf92fe7f1b40a4256f0ec529f444f"><code>cornerstonejs/cornerstoneWADOImageLoader</code></a></h3>
<p><strong>Message:</strong> feat(multiframe): fix the build for the multiframe (#524)</p>
<p><strong>Author:</strong> Alireza</p>
<p><strong>Date:</strong> 2023-04-01 01:51:37 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstonewadoimageloader-4" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstonewadoimageloader-4"><a href="https://github.com/cornerstonejs/cornerstoneWADOImageLoader/commit/a10e369cbafa980c208ae6eeff00c0c31f0b71f1"><code>cornerstonejs/cornerstoneWADOImageLoader</code></a></h3>
<p><strong>Message:</strong> feat(multiframe): fix the build for multiframe (#523)</p>
<p><strong>Author:</strong> Alireza</p>
<p><strong>Date:</strong> 2023-04-01 01:41:21 UTC</p>
<hr>
</section>
<section id="vjcitnbioconcotk" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnbioconcotk"><a href="https://github.com/vjcitn/BiocOncoTK/commit/b9614f4e3e68275547251fe144c0f2e472f5829d"><code>vjcitn/BiocOncoTK</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘master’ of https://github.com/vjcitn/BiocOncoTK</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2023-02-05 20:26:03 UTC</p>
<hr>
</section>
<section id="vjcitnbioconcotk-1" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnbioconcotk-1"><a href="https://github.com/vjcitn/BiocOncoTK/commit/40b8a2722aa57dcdd9d01241c439c544a0f0a97a"><code>vjcitn/BiocOncoTK</code></a></h3>
<p><strong>Message:</strong> add the Dockerfile for Dockstore</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2023-02-05 20:25:32 UTC</p>
<hr>
</section>
<section id="vjcitnbioconcotk-2" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnbioconcotk-2"><a href="https://github.com/vjcitn/BiocOncoTK/commit/46a6cde356cc0f3f7feb3401116be887e83f6827"><code>vjcitn/BiocOncoTK</code></a></h3>
<p><strong>Message:</strong> updating docker reference</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2023-02-05 18:16:52 UTC</p>
<hr>
</section>
<section id="vjcitnbioconcotk-3" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnbioconcotk-3"><a href="https://github.com/vjcitn/BiocOncoTK/commit/7001d6a2761c929c9688e16d36f034035afbaeb0"><code>vjcitn/BiocOncoTK</code></a></h3>
<p><strong>Message:</strong> adding test parameter file</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2023-02-05 13:58:20 UTC</p>
<hr>
</section>
<section id="vjcitnbioconcotk-4" class="level3">
<h3 class="anchored" data-anchor-id="vjcitnbioconcotk-4"><a href="https://github.com/vjcitn/BiocOncoTK/commit/438fb9deea31b9f41bf4b10355b339d29507f244"><code>vjcitn/BiocOncoTK</code></a></h3>
<p><strong>Message:</strong> fix wdl</p>
<p><strong>Author:</strong> vjcitn</p>
<p><strong>Date:</strong> 2023-02-05 13:42:51 UTC</p>
<hr>
</section>
<section id="cbicacaptk" class="level3">
<h3 class="anchored" data-anchor-id="cbicacaptk"><a href="https://github.com/CBICA/CaPTk/commit/cce88180d3159546a057b32ca9b4172d3ee2624d"><code>CBICA/CaPTk</code></a></h3>
<p><strong>Message:</strong> Merge pull request #1494 from AlexanderGetka-cbica/new-branch-roi-4d</p>
<p><strong>Author:</strong> Alexander Getka</p>
<p><strong>Date:</strong> 2022-05-28 07:23:58 UTC</p>
<hr>
</section>
<section id="cbicacaptk-1" class="level3">
<h3 class="anchored" data-anchor-id="cbicacaptk-1"><a href="https://github.com/CBICA/CaPTk/commit/af70ae7abe3d51e7dc3d024b28cc1117f79bf278"><code>CBICA/CaPTk</code></a></h3>
<p><strong>Message:</strong> Make Sarthak’s suggested edits, re-enable 4D saving</p>
<p><strong>Author:</strong> alexandergetka</p>
<p><strong>Date:</strong> 2022-05-27 19:10:27 UTC</p>
<hr>
</section>
<section id="cbicacaptk-2" class="level3">
<h3 class="anchored" data-anchor-id="cbicacaptk-2"><a href="https://github.com/CBICA/CaPTk/commit/129805d97c1d4e048195718540d8ef9171ad7218"><code>CBICA/CaPTk</code></a></h3>
<p><strong>Message:</strong> 1.9.0 update w/ docs, readme, cmake</p>
<p><strong>Author:</strong> alexandergetka</p>
<p><strong>Date:</strong> 2022-05-27 06:44:06 UTC</p>
<hr>
</section>
<section id="cbicacaptk-3" class="level3">
<h3 class="anchored" data-anchor-id="cbicacaptk-3"><a href="https://github.com/CBICA/CaPTk/commit/ae35ebd68161242d8eae03febaa2531b97703958"><code>CBICA/CaPTk</code></a></h3>
<p><strong>Message:</strong> Fix ROI orientation, especially 4D</p>
<p><strong>Author:</strong> alexandergetka</p>
<p><strong>Date:</strong> 2022-05-27 06:28:22 UTC</p>
<hr>
</section>
<section id="cbicacaptk-4" class="level3">
<h3 class="anchored" data-anchor-id="cbicacaptk-4"><a href="https://github.com/CBICA/CaPTk/commit/33cf03cde8ac8e55bfe74e0e02803ddf26a89e3e"><code>CBICA/CaPTk</code></a></h3>
<p><strong>Message:</strong> Merge pull request #1493 from AlexanderGetka-cbica/add-ibsi2-gui</p>
<p><strong>Author:</strong> Alexander Getka</p>
<p><strong>Date:</strong> 2022-05-06 19:16:33 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone-2" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone-2"><a href="https://github.com/cornerstonejs/cornerstone/commit/56f51a0752955bab8a308bc900b95e28992cf3ea"><code>cornerstonejs/cornerstone</code></a></h3>
<p><strong>Message:</strong> Merge pull request #569 from cornerstonejs/fix/broken-build-circle</p>
<p><strong>Author:</strong> Igor Octaviano</p>
<p><strong>Date:</strong> 2021-11-19 21:46:33 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone-3" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone-3"><a href="https://github.com/cornerstonejs/cornerstone/commit/02e52f2b1f787a145e672e36049efdb12d290e8e"><code>cornerstonejs/cornerstone</code></a></h3>
<p><strong>Message:</strong> fix(circleci): update circleci/node image</p>
<p><strong>Author:</strong> igoroctaviano</p>
<p><strong>Date:</strong> 2021-11-19 21:41:52 UTC</p>
<hr>
</section>
<section id="cornerstonejscornerstone-4" class="level3">
<h3 class="anchored" data-anchor-id="cornerstonejscornerstone-4"><a href="https://github.com/cornerstonejs/cornerstone/commit/2bca56dcfeed9e9cbd74c8399e5b66bee9317930"><code>cornerstonejs/cornerstone</code></a></h3>
<p><strong>Message:</strong> Merge pull request #567 from cornerstonejs/refactor/add-commitzen</p>
<p><strong>Author:</strong> Igor Octaviano</p>
<p><strong>Date:</strong> 2021-11-19 19:11:51 UTC</p>
<hr>
</section>
<section id="waldronlabpublicdataresources-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabpublicdataresources-1"><a href="https://github.com/waldronlab/PublicDataResources/commit/487c33b94a9a2e854070e2649b0909f92d4f21b7"><code>waldronlab/PublicDataResources</code></a></h3>
<p><strong>Message:</strong> magrittr import</p>
<p><strong>Author:</strong> lwaldron</p>
<p><strong>Date:</strong> 2021-04-29 10:43:54 UTC</p>
<hr>
</section>
<section id="waldronlabpublicdataresources-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabpublicdataresources-2"><a href="https://github.com/waldronlab/PublicDataResources/commit/b5bb57e4d0a076867d3ce53237a9af200de64a55"><code>waldronlab/PublicDataResources</code></a></h3>
<p><strong>Message:</strong> RELEASE_3_12, chunk names</p>
<p><strong>Author:</strong> lwaldron</p>
<p><strong>Date:</strong> 2021-04-29 10:22:10 UTC</p>
<hr>
</section>
<section id="waldronlabpublicdataresources-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabpublicdataresources-3"><a href="https://github.com/waldronlab/PublicDataResources/commit/ad9a03fea49394b85cf301cc94904c10fb8689d6"><code>waldronlab/PublicDataResources</code></a></h3>
<p><strong>Message:</strong> use release, names for main sections</p>
<p><strong>Author:</strong> lwaldron</p>
<p><strong>Date:</strong> 2021-04-29 09:58:41 UTC</p>
<hr>
</section>
<section id="vjcitntxreginfra2" class="level3">
<h3 class="anchored" data-anchor-id="vjcitntxreginfra2"><a href="https://github.com/vjcitn/TxRegInfra2/commit/dec9ea9b250e82b70a7ad94fd0119c329d1a6919"><code>vjcitn/TxRegInfra2</code></a></h3>
<p><strong>Message:</strong> revise vignette to use $run</p>
<p><strong>Author:</strong> Vince Carey</p>
<p><strong>Date:</strong> 2020-12-31 01:01:51 UTC</p>
<hr>
</section>
<section id="vjcitntxreginfra2-1" class="level3">
<h3 class="anchored" data-anchor-id="vjcitntxreginfra2-1"><a href="https://github.com/vjcitn/TxRegInfra2/commit/33a997013dcbd9a1b313decb6a32f6bffad07d9b"><code>vjcitn/TxRegInfra2</code></a></h3>
<p><strong>Message:</strong> synch with gh-pages</p>
<p><strong>Author:</strong> Vince Carey</p>
<p><strong>Date:</strong> 2020-12-26 02:26:54 UTC</p>
<hr>
</section>
<section id="vjcitntxreginfra2-2" class="level3">
<h3 class="anchored" data-anchor-id="vjcitntxreginfra2-2"><a href="https://github.com/vjcitn/TxRegInfra2/commit/650bb0ef53a2c260830c1c04ff0131570333170d"><code>vjcitn/TxRegInfra2</code></a></h3>
<p><strong>Message:</strong> update content</p>
<p><strong>Author:</strong> Vince Carey</p>
<p><strong>Date:</strong> 2020-12-26 02:01:30 UTC</p>
<hr>
</section>
<section id="vjcitntxreginfra2-3" class="level3">
<h3 class="anchored" data-anchor-id="vjcitntxreginfra2-3"><a href="https://github.com/vjcitn/TxRegInfra2/commit/ef33f37d1f51b393d02fb5e90ea51a62241e311a"><code>vjcitn/TxRegInfra2</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘main’ into gh-pages – bring in README.md</p>
<p><strong>Author:</strong> Vince Carey</p>
<p><strong>Date:</strong> 2020-12-26 01:41:10 UTC</p>
<hr>
</section>
<section id="vjcitntxreginfra2-4" class="level3">
<h3 class="anchored" data-anchor-id="vjcitntxreginfra2-4"><a href="https://github.com/vjcitn/TxRegInfra2/commit/52342454ecb895ebb61247f41f5fa8632da521a0"><code>vjcitn/TxRegInfra2</code></a></h3>
<p><strong>Message:</strong> update README</p>
<p><strong>Author:</strong> Vince Carey</p>
<p><strong>Date:</strong> 2020-12-26 01:40:38 UTC</p>
<hr>
</section>
<section id="waldronlabenrichomics-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabenrichomics-1"><a href="https://github.com/waldronlab/enrichOmics/commit/3ef2d06d2a00a372aef1e01f3e1f0c5d8ea1c850"><code>waldronlab/enrichOmics</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘master’ of https://github.com/waldronlab/enrichOmics</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2020-09-25 21:15:25 UTC</p>
<hr>
</section>
<section id="waldronlabenrichomics-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabenrichomics-2"><a href="https://github.com/waldronlab/enrichOmics/commit/4ba2deaaebc1195237efe779425c0b2d133c15b3"><code>waldronlab/enrichOmics</code></a></h3>
<p><strong>Message:</strong> post-conference updates</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2020-09-25 21:14:58 UTC</p>
<hr>
</section>
<section id="waldronlabenrichomics-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabenrichomics-3"><a href="https://github.com/waldronlab/enrichOmics/commit/71abc90ea5822907509f6ab3232bc4663d33d10c"><code>waldronlab/enrichOmics</code></a></h3>
<p><strong>Message:</strong> Merge pull request #11 from waldronlab/lwaldron-patch-1</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2020-07-28 17:55:44 UTC</p>
<hr>
</section>
<section id="waldronlabenrichomics-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabenrichomics-4"><a href="https://github.com/waldronlab/enrichOmics/commit/b8d7cd271cb9f57aa8938d1ec8483cb960897f69"><code>waldronlab/enrichOmics</code></a></h3>
<p><strong>Message:</strong> Use bioconductor/bioconductor_docker:bioc2020.1</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2020-07-28 17:48:35 UTC</p>
<hr>
</section>
<section id="waldronlabpublicdataresources-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabpublicdataresources-4"><a href="https://github.com/waldronlab/PublicDataResources/commit/7da876a4527226e5c7a3011fbd0dc84297d0b603"><code>waldronlab/PublicDataResources</code></a></h3>
<p><strong>Message:</strong> add link to slides</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2020-07-28 07:42:23 UTC</p>
<hr>
</section>
<section id="seandaviawesome-cancer-variant-resources-4" class="level3">
<h3 class="anchored" data-anchor-id="seandaviawesome-cancer-variant-resources-4"><a href="https://github.com/seandavi/awesome-cancer-variant-resources/commit/cb2516696e522d14f06be35a3c635da37df39588"><code>seandavi/awesome-cancer-variant-resources</code></a></h3>
<p><strong>Message:</strong> Merge pull request #3 from MuhammedHasan/patch-1</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2019-03-13 12:05:47 UTC</p>
<hr>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>