<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>U24CancerProjects with Topics: u24ca289073 or nci-itcr</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-2f5df379a58b258e96c21c0638c20c03.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-d1b12f2568ecbe55642fee6aa00bd082.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="style.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">U24CancerProjectHub</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./overview.html"> 
<span class="menu-text">Overview</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./projects.html" aria-current="page"> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./commit_log.html"> 
<span class="menu-text">CommitLog</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content column-page" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">U24CancerProjects with Topics: u24ca289073 or nci-itcr</h1>
<p class="subtitle lead">A list of repositories discovered on GitHub. The table is sortable by clicking on headers and searchable via the search box.</p>
</div>



<div class="quarto-title-meta column-page">

    
  
    
  </div>
  


</header>


<style>.dataframe {
width: 100%;
border-collapse: collapse;
margin-top: 20px;
}
.dataframe th, .dataframe td {
border: 1px solid #ddd;
padding: 8px;
text-align: left;
}
.dataframe th {
background-color: #f2f2f2;
cursor: pointer;
}
</style>
<p><link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.11.5/css/jquery.dataTables.css"></p>
<table class="dataframe table table-sm table-striped small">
<thead>
<tr>
<th>
Repository
</th>
<th>
Owner
</th>
<th>
Description
</th>
<th>
Stars
</th>
<th>
Last Updated
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
awesome-cancer-variant-resources<a href="https://github.com/seandavi/awesome-cancer-variant-resources" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
seandavi
</td>
<td>
A community-maintained repository of cancer clinical knowledge bases and databases focused on cancer variants.
</td>
<td>
315
</td>
<td>
2025-03-06
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html" target="_blank" style="color: red;">MultiAssayExperiment</a><a href="https://github.com/waldronlab/MultiAssayExperiment" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
Bioconductor package for management of multi-assay data
</td>
<td>
73
</td>
<td>
2025-08-08
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/BiocParallel.html" target="_blank" style="color: red;">BiocParallel</a><a href="https://github.com/Bioconductor/BiocParallel" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocParallel.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
Bioconductor facilities for parallel evaluation
</td>
<td>
68
</td>
<td>
2025-07-18
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/SpatialExperiment.html" target="_blank" style="color: red;">SpatialExperiment</a><a href="https://github.com/drighelli/SpatialExperiment" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/SpatialExperiment.html" target="_blank"> </a>
</td>
<td>
drighelli
</td>
<td>
</td>
<td>
66
</td>
<td>
2025-05-28
</td>
</tr>
<tr>
<td>
HGNChelper<a href="https://github.com/waldronlab/HGNChelper" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
waldronlab
</td>
<td>
Identify and correct invalid gene symbols
</td>
<td>
58
</td>
<td>
2024-11-16
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/data/experiment/html/curatedTCGAData.html" target="_blank" style="color: red;">curatedTCGAData</a><a href="https://github.com/waldronlab/curatedTCGAData" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/data/experiment/html/curatedTCGAData.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
Curated Data From The Cancer Genome Atlas (TCGA) as MultiAssayExperiment Objects
</td>
<td>
44
</td>
<td>
2025-07-21
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/cBioPortalData.html" target="_blank" style="color: red;">cBioPortalData</a><a href="https://github.com/waldronlab/cBioPortalData" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/cBioPortalData.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
Integrate the cancer genomics portal, cBioPortal, using MultiAssayExperiment
</td>
<td>
33
</td>
<td>
2025-07-21
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/TCGAutils.html" target="_blank" style="color: red;">TCGAutils</a><a href="https://github.com/waldronlab/TCGAutils" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/TCGAutils.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
Toolbox package for organizing and working with TCGA data
</td>
<td>
28
</td>
<td>
2025-07-14
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/DelayedArray.html" target="_blank" style="color: red;">DelayedArray</a><a href="https://github.com/Bioconductor/DelayedArray" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/DelayedArray.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
A unified framework for working transparently with on-disk and in-memory array-like datasets
</td>
<td>
27
</td>
<td>
2025-07-11
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/BiocPkgTools.html" target="_blank" style="color: red;">BiocPkgTools</a><a href="https://github.com/seandavi/BiocPkgTools" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocPkgTools.html" target="_blank"> </a>
</td>
<td>
seandavi
</td>
<td>
Computable build reports, package metadata, and download stats from the Bioconductor project
</td>
<td>
21
</td>
<td>
2025-07-15
</td>
</tr>
<tr>
<td>
MultiAssayWorkshop<a href="https://github.com/waldronlab/MultiAssayWorkshop" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
waldronlab
</td>
<td>
Multi-omic Integration and Analysis of cBioPortal and TCGA data with MultiAssayExperiment
</td>
<td>
21
</td>
<td>
2025-07-02
</td>
</tr>
<tr>
<td>
enrichOmics<a href="https://github.com/waldronlab/enrichOmics" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
waldronlab
</td>
<td>
Functional enrichment analysis of high-throughput omics data
</td>
<td>
20
</td>
<td>
2025-01-06
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/devel/bioc/html/AnnotationHub.html" target="_blank" style="color: red;">AnnotationHub</a><a href="https://github.com/Bioconductor/AnnotationHub" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/devel/bioc/html/AnnotationHub.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
Client for the Bioconductor AnnotationHub web resource
</td>
<td>
19
</td>
<td>
2025-07-25
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/data/experiment/html/SingleCellMultiModal.html" target="_blank" style="color: red;">SingleCellMultiModal</a><a href="https://github.com/waldronlab/SingleCellMultiModal" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/data/experiment/html/SingleCellMultiModal.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
Single Cell multimodal data scripts for downloading datasets
</td>
<td>
17
</td>
<td>
2025-06-12
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/GenomicSuperSignature.html" target="_blank" style="color: red;">GenomicSuperSignature</a><a href="https://github.com/shbrief/GenomicSuperSignature" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/GenomicSuperSignature.html" target="_blank"> </a>
</td>
<td>
shbrief
</td>
<td>
Interpretation of RNAseq experiments through robust, efficient comparison to public databases
</td>
<td>
16
</td>
<td>
2025-07-31
</td>
</tr>
<tr>
<td>
PublicDataResources<a href="https://github.com/waldronlab/PublicDataResources" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
waldronlab
</td>
<td>
Public data resources and Bioconductor: The goal of this workshop is to introduce Bioconductor packages for finding, accessing, and using large-scale public data resources including the Gene Expression Omnibus GEO, Sequence Read Archive SRA, the Genomic Data Commons GDC, and Bioconductor-hosted curated data resources for metagenomics, pharmacogenomics PharmacoDB, and The Cancer Genome Atlas.
</td>
<td>
14
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/GSEABenchmarkeR.html" target="_blank" style="color: red;">GSEABenchmarkeR</a><a href="https://github.com/waldronlab/GSEABenchmarkeR" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/GSEABenchmarkeR.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
Reproducible GSEA Benchmarking
</td>
<td>
13
</td>
<td>
2025-05-08
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/BiocFileCache.html" target="_blank" style="color: red;">BiocFileCache</a><a href="https://github.com/Bioconductor/BiocFileCache" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/BiocFileCache.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
Manage Files Across Sessions
</td>
<td>
13
</td>
<td>
2025-07-23
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/HDF5Array.html" target="_blank" style="color: red;">HDF5Array</a><a href="https://github.com/Bioconductor/HDF5Array" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/HDF5Array.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
HDF5 datasets as array-like objects in R
</td>
<td>
12
</td>
<td>
2025-05-28
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/data/experiment/html/curatedOvarianData.html" target="_blank" style="color: red;">curatedOvarianData</a><a href="https://github.com/waldronlab/curatedOvarianData" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/data/experiment/html/curatedOvarianData.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
The curatedOvarianData package provides data for gene expression analysis in patients with ovarian cancer
</td>
<td>
11
</td>
<td>
2025-05-05
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/ExperimentHub.html" target="_blank" style="color: red;">ExperimentHub</a><a href="https://github.com/Bioconductor/ExperimentHub" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/ExperimentHub.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
Client to access ExperimentHub resources
</td>
<td>
11
</td>
<td>
2025-07-17
</td>
</tr>
<tr>
<td>
BiocOncoTK<a href="https://github.com/vjcitn/BiocOncoTK" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
vjcitn
</td>
<td>
Bioconductor components for general cancer genomics
</td>
<td>
10
</td>
<td>
2023-02-05
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/devel/bioc/html/CNVRanger.html" target="_blank" style="color: red;">CNVRanger</a><a href="https://github.com/waldronlab/CNVRanger" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/devel/bioc/html/CNVRanger.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
Functionality for CNV analysis
</td>
<td>
9
</td>
<td>
2025-05-08
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/devel/bioc/html/SparseArray.html" target="_blank" style="color: red;">SparseArray</a><a href="https://github.com/Bioconductor/SparseArray" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/devel/bioc/html/SparseArray.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
High-performance sparse data representation and manipulation in R
</td>
<td>
9
</td>
<td>
2025-07-18
</td>
</tr>
<tr>
<td>
MultiAssayExperiment.TCGA<a href="https://github.com/waldronlab/MultiAssayExperiment.TCGA" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
waldronlab
</td>
<td>
The pipeline to produce TCGA data as MultiAssayExperiment objects
</td>
<td>
7
</td>
<td>
2023-05-16
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/S4Arrays.html" target="_blank" style="color: red;">S4Arrays</a><a href="https://github.com/Bioconductor/S4Arrays" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/S4Arrays.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
Foundation of array-like containers in Bioconductor
</td>
<td>
6
</td>
<td>
2025-05-28
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/RaggedExperiment.html" target="_blank" style="color: red;">RaggedExperiment</a><a href="https://github.com/Bioconductor/RaggedExperiment" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/RaggedExperiment.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
Matrix-like representations of mutation and CN data
</td>
<td>
5
</td>
<td>
2025-07-10
</td>
</tr>
<tr>
<td>
scConform<a href="https://github.com/ccb-hms/scConform" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
ccb-hms
</td>
<td>
Uncertainty quantification for cell type annotation using conformal inference
</td>
<td>
4
</td>
<td>
2024-07-25
</td>
</tr>
<tr>
<td>
OmicsMLRepoData<a href="https://github.com/waldronlab/OmicsMLRepoData" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
waldronlab
</td>
<td>
Translate Bioconductor resources into AI/ML-ready format + Manual curation/harmonization of the metadata for improved usability
</td>
<td>
3
</td>
<td>
2025-07-26
</td>
</tr>
<tr>
<td>
state-cancer-profile-scraper<a href="https://github.com/seandavi/state-cancer-profile-scraper" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
seandavi
</td>
<td>
Bulk access and download state cancer profiles data
</td>
<td>
3
</td>
<td>
2025-08-01
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/OmicsMLRepoR.html" target="_blank" style="color: red;">OmicsMLRepoR</a><a href="https://github.com/shbrief/OmicsMLRepoR" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/OmicsMLRepoR.html" target="_blank"> </a>
</td>
<td>
shbrief
</td>
<td>
Bioconductor package to access the harmonized metadata through ontology
</td>
<td>
2
</td>
<td>
2025-07-26
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/GenomicFiles.html" target="_blank" style="color: red;">GenomicFiles</a><a href="https://github.com/Bioconductor/GenomicFiles" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/GenomicFiles.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
Distributed computing by file or by range
</td>
<td>
2
</td>
<td>
2025-07-21
</td>
</tr>
<tr>
<td>
YESCDS<a href="https://github.com/vjcitn/YESCDS" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
vjcitn
</td>
<td>
Cancer Data Science resources for DFCI YES for CURE
</td>
<td>
2
</td>
<td>
2024-10-30
</td>
</tr>
<tr>
<td>
refinebioc<a href="https://github.com/seandavi/refinebioc" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
seandavi
</td>
<td>
The bridge between refine.bio and Bioconductor
</td>
<td>
2
</td>
<td>
2025-03-05
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/h5mread.html" target="_blank" style="color: red;">h5mread</a><a href="https://github.com/Bioconductor/h5mread" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/h5mread.html" target="_blank"> </a>
</td>
<td>
Bioconductor
</td>
<td>
A fast HDF5 reader
</td>
<td>
1
</td>
<td>
2025-05-28
</td>
</tr>
<tr>
<td>
BiocBuildDB<a href="https://github.com/seandavi/BiocBuildDB" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
seandavi
</td>
<td>
Longitudinal Bioc build database
</td>
<td>
1
</td>
<td>
2025-03-20
</td>
</tr>
<tr>
<td>
oncoKBData<a href="https://github.com/waldronlab/oncoKBData" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
waldronlab
</td>
<td>
Expose the Onco KB API via R / Bioconductor
</td>
<td>
1
</td>
<td>
2025-05-09
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/xenLite.html" target="_blank" style="color: red;">xenLite</a><a href="https://github.com/vjcitn/xenLite" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/xenLite.html" target="_blank"> </a>
</td>
<td>
vjcitn
</td>
<td>
Lightweight approach to managing Xenium experiments
</td>
<td>
1
</td>
<td>
2024-11-27
</td>
</tr>
<tr>
<td>
BiocHubsIngestR<a href="https://github.com/Bioconductor/BiocHubsIngestR" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
Bioconductor
</td>
<td>
Helper package for users to upload data to temporary BiocHubsIngest endpoints
</td>
<td>
1
</td>
<td>
2025-02-27
</td>
</tr>
<tr>
<td>
TxRegInfra2<a href="https://github.com/vjcitn/TxRegInfra2" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
vjcitn
</td>
<td>
Software for exploring transcriptional regulatory networks, currently using MongoDB documents for various genomic feature types
</td>
<td>
1
</td>
<td>
2020-12-31
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/bedbaser.html" target="_blank" style="color: red;">bedbaser</a><a href="https://github.com/waldronlab/bedbaser" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/bedbaser.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
An R client for bedbase.org
</td>
<td>
1
</td>
<td>
2025-07-22
</td>
</tr>
<tr>
<td>
curatedMetagenomicDataODM<a href="https://github.com/shbrief/curatedMetagenomicDataODM" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
shbrief
</td>
<td>
Genestack's Open Data Management (ODM) setup for curatedMetagenomicData curation
</td>
<td>
1
</td>
<td>
2025-07-28
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/XeniumIO.html" target="_blank" style="color: red;">XeniumIO</a><a href="https://github.com/waldronlab/XeniumIO" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/XeniumIO.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
software for importing 10X Xenium data
</td>
<td>
0
</td>
<td>
2025-07-03
</td>
</tr>
<tr>
<td>
programets<a href="https://github.com/nih-cfde/programets" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
nih-cfde
</td>
<td>
</td>
<td>
0
</td>
<td>
2025-07-28
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/TENxIO.html" target="_blank" style="color: red;">TENxIO</a><a href="https://github.com/waldronlab/TENxIO" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/TENxIO.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
Bioconductor interface to 10X files
</td>
<td>
0
</td>
<td>
2025-05-08
</td>
</tr>
<tr>
<td>
hovernethelp<a href="https://github.com/hpages/hovernethelp" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
hpages
</td>
<td>
</td>
<td>
0
</td>
<td>
2025-05-22
</td>
</tr>
<tr>
<td>
HGNCREST<a href="https://github.com/waldronlab/HGNCREST" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
waldronlab
</td>
<td>
Query the HGNC REST API with R
</td>
<td>
0
</td>
<td>
2025-05-09
</td>
</tr>
<tr>
<td>
community-bioc-archive<a href="https://github.com/waldronlab/community-bioc-archive" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
waldronlab
</td>
<td>
Text archive of community-bioc Slack public channels
</td>
<td>
0
</td>
<td>
2025-05-02
</td>
</tr>
<tr>
<td>
<a href="https://www.bioconductor.org/packages/release/bioc/html/VisiumIO.html" target="_blank" style="color: red;">VisiumIO</a><a href="https://github.com/waldronlab/VisiumIO" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://www.bioconductor.org/packages/release/bioc/html/VisiumIO.html" target="_blank"> </a>
</td>
<td>
waldronlab
</td>
<td>
Import spaceranger output and 10X spatial data
</td>
<td>
0
</td>
<td>
2025-05-07
</td>
</tr>
<tr>
<td>
u24Project<a href="https://github.com/bd4everyone/u24Project" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
bd4everyone
</td>
<td>
Tracking for project 1U24CA289073-01
</td>
<td>
0
</td>
<td>
2025-07-08
</td>
</tr>
<tr>
<td>
Viewers<a href="https://github.com/OHIF/Viewers" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
OHIF
</td>
<td>
OHIF zero-footprint DICOM viewer and oncology specific Lesion Tracker, plus shared extension packages
</td>
<td>
3763
</td>
<td>
2025-08-11
</td>
</tr>
<tr>
<td>
cornerstone<a href="https://github.com/cornerstonejs/cornerstone" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
cornerstonejs
</td>
<td>
[Deprecated] Use Cornerstone3D Instead https://cornerstonejs.org/
</td>
<td>
2098
</td>
<td>
2024-03-07
</td>
</tr>
<tr>
<td>
pyradiomics<a href="https://github.com/AIM-Harvard/pyradiomics" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
AIM-Harvard
</td>
<td>
Open-source python package for the extraction of Radiomics features from 2D and 3D images and binary masks. Support: https://discourse.slicer.org/c/community/radiomics
</td>
<td>
1289
</td>
<td>
2025-08-12
</td>
</tr>
<tr>
<td>
cornerstone3D<a href="https://github.com/cornerstonejs/cornerstone3D" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
cornerstonejs
</td>
<td>
Cornerstone is a set of JavaScript libraries that can be used to build web-based medical imaging applications. It provides a framework to build radiology applications such as the OHIF Viewer.
</td>
<td>
780
</td>
<td>
2025-08-11
</td>
</tr>
<tr>
<td>
dicomParser<a href="https://github.com/cornerstonejs/dicomParser" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
cornerstonejs
</td>
<td>
JavaScript parser for DICOM Part 10 data
</td>
<td>
738
</td>
<td>
2024-05-16
</td>
</tr>
<tr>
<td>
cornerstoneTools<a href="https://github.com/cornerstonejs/cornerstoneTools" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
cornerstonejs
</td>
<td>
[Deprecated] Use Cornerstone3D Instead https://cornerstonejs.org/
</td>
<td>
588
</td>
<td>
2024-07-15
</td>
</tr>
<tr>
<td>
dcmjs<a href="https://github.com/dcmjs-org/dcmjs" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
dcmjs-org
</td>
<td>
Javascript implementation of DICOM manipulation
</td>
<td>
329
</td>
<td>
2025-08-11
</td>
</tr>
<tr>
<td>
cornerstoneWADOImageLoader<a href="https://github.com/cornerstonejs/cornerstoneWADOImageLoader" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
cornerstonejs
</td>
<td>
[Deprecated] Use Cornerstone3D Instead https://cornerstonejs.org/
</td>
<td>
286
</td>
<td>
2023-12-15
</td>
</tr>
<tr>
<td>
dcmqi<a href="https://github.com/QIICR/dcmqi" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
QIICR
</td>
<td>
dcmqi (DICOM for Quantitative Imaging) is a C++ library for conversion between imaging research formats and the standard DICOM representation for image analysis results
</td>
<td>
262
</td>
<td>
2025-07-17
</td>
</tr>
<tr>
<td>
CaPTk<a href="https://github.com/CBICA/CaPTk" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
CBICA
</td>
<td>
Cancer Imaging Phenomics Toolkit (CaPTk) is a software platform to perform image analysis and predictive modeling tasks. Documentation: https://cbica.github.io/CaPTk
</td>
<td>
191
</td>
<td>
2023-12-09
</td>
</tr>
<tr>
<td>
dicomweb-client<a href="https://github.com/dcmjs-org/dicomweb-client" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
dcmjs-org
</td>
<td>
JavaScript client for DICOMweb RESTful services
</td>
<td>
153
</td>
<td>
2025-08-06
</td>
</tr>
<tr>
<td>
react-vtkjs-viewport<a href="https://github.com/OHIF/react-vtkjs-viewport" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
OHIF
</td>
<td>
VTK.js image viewport component in React
</td>
<td>
147
</td>
<td>
2023-09-22
</td>
</tr>
<tr>
<td>
SlicerRadiomics<a href="https://github.com/AIM-Harvard/SlicerRadiomics" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
AIM-Harvard
</td>
<td>
A Slicer extension to provide a GUI around pyradiomics
</td>
<td>
114
</td>
<td>
2024-04-25
</td>
</tr>
<tr>
<td>
dicomweb-server<a href="https://github.com/dcmjs-org/dicomweb-server" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
dcmjs-org
</td>
<td>
Lightweight DICOMweb Server with CouchDB
</td>
<td>
106
</td>
<td>
2024-06-03
</td>
</tr>
<tr>
<td>
react-cornerstone-viewport<a href="https://github.com/cornerstonejs/react-cornerstone-viewport" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
cornerstonejs
</td>
<td>
[Deprecated] Use Cornerstone3D Instead https://cornerstonejs.org/
</td>
<td>
96
</td>
<td>
2024-12-10
</td>
</tr>
<tr>
<td>
react-viewerbase<a href="https://github.com/OHIF/react-viewerbase" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
OHIF
</td>
<td>
Core medical image viewer components built using React
</td>
<td>
87
</td>
<td>
2019-08-14
</td>
</tr>
<tr>
<td>
cornerstoneWebImageLoader<a href="https://github.com/cornerstonejs/cornerstoneWebImageLoader" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
cornerstonejs
</td>
<td>
Cornerstone Image Loader for Web Images (PNG, JPEG)
</td>
<td>
69
</td>
<td>
2023-06-23
</td>
</tr>
<tr>
<td>
cornerstoneMath<a href="https://github.com/cornerstonejs/cornerstoneMath" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
cornerstonejs
</td>
<td>
Math and computation geometry functionality for cornerstone
</td>
<td>
52
</td>
<td>
2023-03-01
</td>
</tr>
<tr>
<td>
civic-client<a href="https://github.com/griffithlab/civic-client" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
griffithlab
</td>
<td>
Web client for CIViC: Clinical Interpretations of Variants in Cancer
</td>
<td>
50
</td>
<td>
2023-11-28
</td>
</tr>
<tr>
<td>
civic-server<a href="https://github.com/griffithlab/civic-server" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
griffithlab
</td>
<td>
Backend Server for CIViC Project
</td>
<td>
39
</td>
<td>
2023-07-06
</td>
</tr>
<tr>
<td>
VTKPlugin<a href="https://github.com/OHIF/VTKPlugin" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
OHIF
</td>
<td>
OHIF Plugin for The Visualization Toolkit (VTK)
</td>
<td>
35
</td>
<td>
2018-10-03
</td>
</tr>
<tr>
<td>
cancer<a href="https://github.com/crowds-cure/cancer" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
crowds-cure
</td>
<td>
Crowd-sourcing annotations of medical images to advance cancer research
</td>
<td>
30
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
ohif-core<a href="https://github.com/OHIF/ohif-core" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
OHIF
</td>
<td>
Useful functions and classes for building web-based medical imaging applications
</td>
<td>
25
</td>
<td>
2019-08-14
</td>
</tr>
<tr>
<td>
QuantitativeReporting<a href="https://github.com/QIICR/QuantitativeReporting" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
QIICR
</td>
<td>
Segmentation-based measurements with DICOM import and export of the results.
</td>
<td>
25
</td>
<td>
2025-06-10
</td>
</tr>
<tr>
<td>
civic-v2<a href="https://github.com/griffithlab/civic-v2" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
griffithlab
</td>
<td>
CIViC is an open access, open source, community-driven web resource for Clinical Interpretation of Variants in Cancer
</td>
<td>
24
</td>
<td>
2025-08-04
</td>
</tr>
<tr>
<td>
calculate-suv<a href="https://github.com/cornerstonejs/calculate-suv" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
cornerstonejs
</td>
<td>
Tiny library for calculating Standardized Uptake Value for Nuclear Medicine (i.e.&nbsp;PET, SPECT)
</td>
<td>
23
</td>
<td>
2023-09-08
</td>
</tr>
<tr>
<td>
NG-CHM<a href="https://github.com/MD-Anderson-Bioinformatics/NG-CHM" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
MD-Anderson-Bioinformatics
</td>
<td>
A dynamic, graphical environment for exploration of clustered or non-clustered heat map data in a web browser.
</td>
<td>
11
</td>
<td>
2025-08-10
</td>
</tr>
<tr>
<td>
NGCHM-R<a href="https://github.com/MD-Anderson-Bioinformatics/NGCHM-R" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
MD-Anderson-Bioinformatics
</td>
<td>
An R package for creating Next-Generation Clustered Heat Maps (NG-CHM)
</td>
<td>
10
</td>
<td>
2025-07-07
</td>
</tr>
<tr>
<td>
restfulSE<a href="https://github.com/shwetagopaul92/restfulSE" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
shwetagopaul92
</td>
<td>
demonstration of SummarizedExperiment with assay component retrieved from remote HDF5 server
</td>
<td>
10
</td>
<td>
2022-10-06
</td>
</tr>
<tr>
<td>
ontoProcPRE2021<a href="https://github.com/vjcitn/ontoProcPRE2021" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
vjcitn
</td>
<td>
Ontology processing for Bioconductor
</td>
<td>
9
</td>
<td>
2021-03-15
</td>
</tr>
<tr>
<td>
SlicerDevelopmentToolbox<a href="https://github.com/QIICR/SlicerDevelopmentToolbox" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
QIICR
</td>
<td>
API for 3D Slicer extension development providing widgets, mixins, helpers, decorators etc.
</td>
<td>
7
</td>
<td>
2024-04-23
</td>
</tr>
<tr>
<td>
dicom4miccai-handson<a href="https://github.com/QIICR/dicom4miccai-handson" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
QIICR
</td>
<td>
Supporting materials for the MICCAI 2017 DICOM tutorial. See accompanying documentation at https://qiicr.gitbook.io/dicom4miccai-handson/
</td>
<td>
5
</td>
<td>
2019-06-05
</td>
</tr>
<tr>
<td>
SlicerPathology<a href="https://github.com/SBU-BMI/SlicerPathology" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
SBU-BMI
</td>
<td>
3D Slicer extension for Pathology segmentation tools
</td>
<td>
5
</td>
<td>
2018-05-24
</td>
</tr>
<tr>
<td>
dcm2tables<a href="https://github.com/QIICR/dcm2tables" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
QIICR
</td>
<td>
</td>
<td>
4
</td>
<td>
2019-07-18
</td>
</tr>
<tr>
<td>
<a href="https://bioconductor.org/packages/release/bioc/html/pogos.html" target="_blank" style="color: red;">pogos</a><a href="https://github.com/vjcitn/pogos" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a><a href="https://bioconductor.org/packages/release/bioc/html/pogos.html" target="_blank"> </a>
</td>
<td>
vjcitn
</td>
<td>
PharmacOGenomics Ontology Support
</td>
<td>
3
</td>
<td>
2023-02-05
</td>
</tr>
<tr>
<td>
pixi-plugin<a href="https://github.com/preclinical-imaging/pixi-plugin" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
preclinical-imaging
</td>
<td>
PIXI is an XNAT plugin designed to help manage and analyze preclinical imaging data.
</td>
<td>
2
</td>
<td>
2025-08-04
</td>
</tr>
<tr>
<td>
MultiOmicQC<a href="https://github.com/Bioconductor/MultiOmicQC" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
Bioconductor
</td>
<td>
Quality Control Measures for MultiOmics Experiments via MultiAssayExperiment
</td>
<td>
2
</td>
<td>
2023-03-08
</td>
</tr>
<tr>
<td>
pixi-mice-image-splitter<a href="https://github.com/preclinical-imaging/pixi-mice-image-splitter" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
preclinical-imaging
</td>
<td>
PIXI Mice Hotel Image Splitter
</td>
<td>
2
</td>
<td>
2024-12-10
</td>
</tr>
<tr>
<td>
ivygapSEOLD<a href="https://github.com/vjcitn/ivygapSEOLD" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
vjcitn
</td>
<td>
SummarizedExperiment for Ivy-GAP
</td>
<td>
2
</td>
<td>
2017-10-24
</td>
</tr>
<tr>
<td>
OpenLayersPlugin<a href="https://github.com/OHIF/OpenLayersPlugin" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
OHIF
</td>
<td>
Example OHIF plugin based using OpenLayers
</td>
<td>
1
</td>
<td>
2018-11-21
</td>
</tr>
<tr>
<td>
TxRegInfraObsolete<a href="https://github.com/vjcitn/TxRegInfraObsolete" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
vjcitn
</td>
<td>
Some Bioconductor-oriented infrastructure for TxRegQuery
</td>
<td>
1
</td>
<td>
2019-07-04
</td>
</tr>
<tr>
<td>
dicom4miccai<a href="https://github.com/QIICR/dicom4miccai" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
QIICR
</td>
<td>
Web site for the DICOM4MICCAI tutorial at the MICCAI conference (2017 and 2018)
</td>
<td>
1
</td>
<td>
2018-11-14
</td>
</tr>
<tr>
<td>
patelGBMSC<a href="https://github.com/vjcitn/patelGBMSC" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
vjcitn
</td>
<td>
Lightly curated version of single-cell RNA-seq quantifications from CONQUER for PMID 24925914
</td>
<td>
0
</td>
<td>
2018-02-25
</td>
</tr>
<tr>
<td>
restfulSEData<a href="https://github.com/shwetagopaul92/restfulSEData" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
shwetagopaul92
</td>
<td>
</td>
<td>
0
</td>
<td>
2018-05-03
</td>
</tr>
<tr>
<td>
docker-dcmtk-cli<a href="https://github.com/QIICR/docker-dcmtk-cli" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
QIICR
</td>
<td>
Repository to keep Dockerfiles for automated Docker builds of DCMTK
</td>
<td>
0
</td>
<td>
2017-05-16
</td>
</tr>
<tr>
<td>
CRAVAT<a href="https://github.com/KarchinLab/CRAVAT" target="_blank"><img src="githubicon.svg" style="height: 0.9em; vertical-align: middle; margin-left: 10px; margin-right: 10px"></a>
</td>
<td>
KarchinLab
</td>
<td>
</td>
<td>
0
</td>
<td>
2017-06-29
</td>
</tr>
</tbody>
</table>
<script type="text/javascript" charset="utf8" src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.11.5/js/jquery.dataTables.js"></script>
<script>
$(document).ready( function () { 
$('.dataframe').DataTable();
} );
</script>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "î§‹";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>